zjjX*mjXiL~!KnOYDZt5ynhWz4s0%uFiNyU@QpU8hwYyd@p%&-`>uFX#D`)y_JJHS=
zFbB^eH4pU#CJ?vSO20$n6N5shn30A#AdH+n7XoO$ej9ods1G06#g7Dk(|WIj>`>k~
zb)M8wAo;byA8xg0HqZI>z2<3yGfWm`I7_*5<<KOO#L$edv4Ze758X*@p!R9Y!4l-|
zFVSH({Zcd9es;)+>P0~5+AzRcLUC2Zj77!}Jw~~dmIElz0r(yw(F=v={=c;E&IBQG
zaCkA-IsAf6ukoxT47&Z^GE(htDLztvc8_~)ljVz0Htr7XXfO@>`4g$Bk0*jKevyY`
zTdWN84%9)&tvronOfaJ&=RcFOw)mCL#p0&@b^ogXsaON30xF=kKKbet!oXT-@Zn1V
z4K-IhA5i_W!2_F|3K?HHF(|_9+fXjbw%O0#E7%P#X^fW{zZ(EP2+H6z#^dHMzZPN2
z0KgJ|?CLMLvV?N9Cq)XSIe+bdU#eu`H{rL`l=^V7TwIyi)~DaEE_JdmNiWQh4z>#m
z)Bqu;@cIc+q-p}6X<+<ZmVORQN}7K+wLl5D?X{Ar%fB$}M~dZ<v)WI7YL^BOYMcbk
z6yv+%$;B;mrmG(KmHaB;VR^zFcVM_m3njB^|LN3sT{sPy6jXjE;HfLE-d&>Iy}Xf5
zS{%ujgbx!pnmN*@@q{jJU2w~N6gEz@ta)gK5GkJSCmD5IO$JN<Gl>?a>icbp{^Mg}
zn-}rmBzJ=QTz<fd4<$uL%4?$erc}ZxBizRYw)aUYwk=0KC;E?{3Q2|w=%+r><29zE
zU*+}}mgpI};ip%shxj%WWR@A54De9nRR;>kM0s31r)f$2HBo8^!}JTzsNJPKJeh2*
zI0V$6`cQ$N5w^PC@GVO<BB<E$rWAHLf?qfWJ!2%&rTI+k@br2<cd^XZNy$;8y=%v)
zu&<J~Iq#O*H(gCS&su^kUw(sAK&wEB<1Me(1!DFzcDB=W7lnY^fka^>O<UF-E2#@k
zXrhfu?m88vSLo8`oug^_rUB{5rG~P8g7)GsIgobSaWJ9KVy*giQWONgq3p07IdELp
zmVX!90nm=#vA$BLZsTm$=2_Ke#9bwQUS4>d+4oV_%Nv5@k~JO}bs}Opu){x1oB6sx
z-xl4JwlU3cv4eE(m4LOlL`jCt14Z1RjnE&G9jF+vr3UlDlz(eHYQb)<4GbudJ0-0)
z!I%w!(R~6)pGnDaNN;tjZqRjf&PiL<l!AopdW<qdnvlzU&OFUhhM$zehL8gY53i!6
z7tDk|eaOB6%n=BVcR4v9*fVM)W=Y{(?}gYwr!8w{7flIxn#D15Q{H{g%*oK6Ew7dO
z6sDg^F>#zl+UZ)$SVz*<yXh?!qLYoExh>@s2^;STxACO)j@3%n)humNraIG_g%5!p
zdIW$72#eIfV}1c|@Dg*vRA+IB+xGyDhoBeDEog;xyuP$H#*i?c!)>*s%X*LIU1mym
zJ<h@_!uODG6eWo<HfNt-fK9VxJq=bFOwfb3!4*s!wLntwTo@y#VkP>8XhSqs?L##K
zgcplObK`EeT$zJ*-jp#E8MkJYdW>9bVq91^PC}5Y4k`#s^R~BmcUDPloQ#DK4f6{j
zQ27{9Gu4^!_d%9=4=!Q?@+t<M(C;03KIj@ZR~~~}TlqsSO2aI0O>PaDj$p6;!7-Ep
z`GmJJ;rK2kUjDW(JJ}H>0=U!9vmNOb;FgGp>2PV^@(WefV`YD0F)u__Iti+!p0@Z|
z7R2J37qcA8it6&%M=#uD;)^EG4lo=Kyu3pOA?SVyhh(0ExQWlN8{%_w$iaJ<41A{P
zQbNP3DiO4E{00AdLn!mp)b;Mot6H3NYFJVfy_6oL4}Q(bdW5l%3aJgX-q?TmSAT)D
zUFPe=g>0Xk8q!2VVG3b!_R?5c`^{E;GLto~=$EiHy$(H0`xWIyMqKdfE{eejB<=vM
zRi?%{SUE8MlucCem<24K8izXxaHx!$5eG!V#7Rq8#g1qar!@A3thp{s5IeGlY<)fA
zeFj(=U3<xBAh4Y5D|nJ<sO}y>j<^a#(wuy|#;&pkMJSl+AU=?Eg`D+S*x6&xR`pHq
zfnsAY8gj_(AEBmiyfv-!8NVFVP-WCL4Z^+VeR1HzUA+lKmC6QnhB{%O5RauukujA+
zFCsMYho^=l3`yup5Bk;J7)q>o=SD99AoRLU2sedGAE`lm%3&%|{e;4u=q657KwJD<
zt@;Z}GY=`r-H$|JSg_*8e5EYZZ{5Ir+Ow(!X)GmR$ew4VDnuP)6bze_g*)qBvVABZ
zaaVPH^HJOBpo12?4!XW5E#V4ptkzgK-&R%=(cUh?Z1Jk|;Q<`BCQ6R+4=Wor_CP6Z
zoW2x)EkarCq1SSDXZtE7GW>;vR%rYHukLL`fO{a8Fex^C`&`1lpTswu&(lXsiUHv+
ziJLz7QW!^~oS2qTo3vzZSsbsM0d{%{+BEg?R5H3x(`0gzjGC-yJ?yo@&r=p9j6fZt
z=S+mVD9(r<M3>R+N&(P$aHud@nzGOiF1z7==fjBVBgi;}Pe2FOOI;<xN9)qP#-A=z
zMnyu*$S4oQF}B!q|Ks#Vw)0it1Nt?72{;5R2d|rl?eY6R?7#ebe7;%ZC@cO6dZRzX
zQ@{MPf$V<;y=wn6DYev9B#Ec{Qj*WGNq~%jl#;{@>C5k=)N=sHZModA_Hx(S;plUR
z!(Gf<-r0gJ_=1^7J?0IUe9I)2;##8m_-`4F_KIVZU8z=ygF~aTvK@Z*9hRfwSX>kf
z%Jr<)hG|9Ll8VHc#U;7=G4x^BRryo1q*bd1i`<%k?z{A@!o`!Qr9IlI6R4el6@%pp
zi6)psFnp(p!iWTzg@VhzEMDBUka)A=Jr^Leg#m+!K~i52*F2o2MewFcLdEaUJFuH*
z(>=@F*NfhRu$R)_7Y6V=Q8bf-7z~P~^56z{@fJ<n<j2V8ymA<a3#RhDY~^=PzeL;8
z0xG3)T;@F8C2~5<Kd3<$>{+2Jh5JEDz#k8s^N%SV9dJ9y`9CkZHBm~f^P&(^0Onv4
zk7cSPb52JI^3Fxfo>?hfD6x&_E3Y7I#7Nm|O=25yXIPq*>z^`AO5bq1Tzx73b@&}*
zzc6ZT(W$xnR?p+M(7dlCIb(xokAxQ9FypdV^b_>Hh1#3b)K{)xbR;CVyVeD=U7Yc$
zFzkU(y+=Q6vRsdq$@*snk5m_RB)2VEC3&1u^{}l@S5;}h4t|W_GvMR8$>2FX!jk(#
zAEK(JE+tV|oO&R@2_$#?0-t_le@y8_-BPv%F_O}keTRX2Bz2OjneC%?H0nC_t!q{U
zmh%GKre0xJ#dpfW;J1zdh5`|f8Ae&W`qjo#Su7?Ha4c;vRS1ANi``}@;@Oa+6(kBt
zkg9;zCrG#J$91Kdg>7on=4W7J!wZ=|3fpRQU@L8i{+3wYqadHDJM#+-B2l=XWZ0$@
zx^<G>UwlR$eh*M0zjlW&N-$#n{R>AIx;o@2DM3#E>){zfh7Bi2xFoNkX8pURV69i>
z@(-BdB8Xwq-Yo+>4^bq9JAb$3I5*c4w+r#%h5>4B{*A>8bvhNTu_E2KY~6)S4~t~n
zOZY3B#(;!Si%t-k&H*$IQEj4XXCw)PWE2-}Gk~D3K<{fFpPl>eaK4i`zM**f`Ch&&
z*(uvsz8EBIVhBHO8YtOvb2jPdd|^jn=ygRy&*%hs0l%-c$;m0<cmx64i2m|HQ+FzS
z^y%$xaY@Cn!J9CL|3&^eGx_`6e|E#BPxDFkFS_I3U9gtijCDW3rwdk9;+91w1nl!R
zN+$xc4k!)`js0}NJ~`4E(i_m-Tp3(fN#E|hb<?}>)MP__8yvKeAFb8p;^Qk+G9*re
z6`86Pg9D?u)(D&mvP`bN95e!z*Y+l!RGY@)7HB~4h^SVrCRsPT<&LmP=8=aI?>l}y
z7z2YSAta`b+v-gb?+wdBqyS0*vJMT7&3TRcaDexw7`ivB{xGut7_Y{Nk_f+^Zp|;8
zDW-e9N~`B18Mct{&gWghVLcwRzYqtShT{5Fz#2nY7QcXFY9JEZ+EX9MKI7pr+Qw8J
zuL4OI$l0FU(cawW$2_Is?D6yu&vXC2kbM>l;4|_0uNTqh`~P-E&erySV}d&`NRC`S
zKPmWGd%}Nxgnwy*ZQJ%=>*t9`3E0^9NKr}T(p3}F6V&3<{pGA8he?pR?Tc;OZU!ch
zVo67zirh#C`Qyw<kz<i9cFZTQ%2<SzEuv+s5a&;aO5LfiTIY}uu{fRv%lIMx%LTmN
z2i|7<v&Fv<0RaAu@BLc{FKO8L)1v@ve}=El9Sz>OuU^zpN0iBduYdq8m`%LDxL+kl
zk-`zl?hR%q-{0MU#sZghNj~hGITO&nBi*nQk2}P+Qd*kl(1w$-QV1u-XHw8((IsaP
zJ#uR2T3MZ*4z4OE$r{X!b+PfXzEsfx!3`l+Fs|rMlr2n@I*jMr6ZTUaNoL+Hq*&np
z+)HQ7`EE=Y{F%NwGS^IAqP8{&LBHJ6YZ6({t$EX5=RJ8r?~%=#GO7JS^S!EkW4vu}
zckHyjlgyZXZgs)NuAeLOg#&M54`yD{l1_7^#d`*C%8tyNlhBUxpiD_kznpRyxgm3~
z^;$Y11+?0w?2EKzUm|d}JzT}W4<U$Q<QmI(PJe@bY-bjSR3?HPc<|Xnb0*Yk^LpnM
zpMr(BWN~qix%9C6i$-(_-oV)}W|f6(6oOq(WTVGI9C$?-0%l*;b7$|14UrFA6YT*=
z<zO|jj(t>b!tA*vSOOK~VkguVGmtnFe-t@>1jTQpAO*`wpxH=5z2VnxeCI(nqW%nZ
z^(mTXqfP?Y7T9{_d@>vDkX`;T9F<A7#F1ngK`KNPSlP6Zq$L}_ramW-ETEZtzArHv
zR**?1R`dQZT_A(`gOdUl0H6dN0D$tJoWS}<#y0jww611WdL~A+?v`fy=c!(DiYDE|
zO^pYeRekrXRZS=gB~{@NbtzCI8kSl{ooHq$XaoF2M1|ob!hsk9_RU5?7N`wl8CV1=
z<&|^@@nQl5``ggE$OO9@5P(4Xq0Bb%se$_>91lWu^jsY;)0|vgTv;4U(^6Az*;@|Y
z-!Grud1c`ik3)1mR~+5SnH~r)Ci31Qw%)9IuM_CB-eT?sb1YpQ7F_TxUC*QVd}g0-
z$`96?=L-lgv){b+eu#=^z0L~~nozQR?>Zh5)#<z$P2Nt?p?s;SPc-9P{8Admkc>li
zr}{%#9&+y_jzBjaO=j`#A)0D==tj}a3OGZy^ZNm)R>X;A+zA^rC!44DHxr%K?F)Qw
z+(Xz?1=nv8_@eeJkR|{&DR+5BEFki(e&Q-eC*g7d%HlT{hdrt;H`~?9*;{aGy?ett
z_!`4#T(s?;I#qUgY!0nhPp~a6A#iG31;D(o)r?@8dV}0!fe|%G+r-(0HG4hy1%&hY
zY7jn-s>P9B%Lf{zzAn5fJsq16`LlaYJv3!&9cqW#>fg2WrZz#*_bO9c!Yj8mwBk8m
zksy8rLhImqbO(KXlwv_j{{;<uuB^a?ejIZG>o9(S1Cyz-U*-Q%qG(N}3pu#<Et)&+
z*PZw;`I;BP*u8@clejf)=y&^zElXD_>X-Ke8xW$b`;dMyR!<kW3u7}ItLDS4cOD|3
z%chss`q!Gx0HTLXulq&;WVZwMs{4Wx=Z4U-@mu_MicRNL;#^SDYn43NDeKoU*O+HC
zO&!3Z<Y2p%1FBdl{r6OD8H?59l31H>{-N*-E3XEXuP^oAagzz&fiuHQc)mdvm3wmu
z%*+qv!9{x|0MHt!IdWq%P?{V=alYS)DvthgT>X(9jQC)iP5oY+F;6{}37gF%9&`s(
z6DGVwJ)0+$ym(2y3qm|a&GTbV7LRu5uyLJfNSy;lHKrX)_A=ctem(uidnimh>X&Aa
z>27Q1QuB|u`T|k(T8d<k(M@=qH23NW{Wb!|{fC;R&t(I%(!r>+6tMm(5e5`!ywvt{
z(uH|IYmZu#;qMp72b~Ir?22c5sdqO~-kQUJ$6}U8F(nXA&Rn9om$gy(-@vVH2vgdO
zX|>Jm*I_J6cP<PMug;%C_>zI^+!DGZG-n)DbA*T^>R_4J3M6Cj=WdgNi!cFgleBZ~
zkrM};dD0OLa1C`fr&f&imP=_PGK0)IvV;47(HodWHylMZmw_#sUI=|_<(k4%uYf_E
z=Kdk!Wo#ZL3{o7;2(fi=dPY;SOuNMj`bxP-i84II)PJ|zO<x2zNjx$TY|R&c%QQ$e
z8pX9}7Y-wPLDG}+g+ToNf?L{$=HT#VmZh@kWJwNC2{*)Z2|)M3u3c@7Xy`*|VMMKm
z*1vp%GcihgQozZVIcGVoZp&yj+;~V*IXUB!!O@jy=)HFRQ3gCRo}piSgV+%D+*(*y
zKmKDW3)(23yrtzi4%QHDQesqi^Jcs4I9pnD^KSTJ@4XeMhp8i)8Wfc1n>m2v0`>vQ
zT}FVV*?@B)3U7V8f~r7tBiwW&e6L9=tpXDfU{xbGp-QSK<f7M$x|l_~V#0i4yLRad
zGjbKSOg?fLxNc~%E6*AjQHSkU(;sFEB2af$t7KBJ@!}uqV8pAAOw}4Nf?av_UXK8e
zj?Qysh<Z~CHaZOXocC7MP(#64<G>4nY3%{<aL^;WxB2mg7P68OqIslY2eXJsKbohZ
zpI^S+6SlSL2FU3Ikyd~BhP%p3){54y*fPvOX;<a5)XlJ>0+cNE7lyvAiuAJbsi|Sf
zJswY!<&|s?Vv;ap-_d#A@d#Q2?8-HgkMi8{au&9nTxg4dgq(wCS#t!9kTgs06e(?z
zNzk~kkFKs<LXDih*`8|G4LOv!@P+%?P5XihRl(8A-ZPx)kJ0!!v|pw)DLWXMfd{1T
z40>weMUl<V%Zq{nb3qB=i18YNZgHX2Q3*56<b%)C;Lagz$|%}jxm5QgHEYd6a!=aV
zaIRjLZa&I1AC!2g+0LWJuRganRj#!edv>IUlQeBeS4^|?fnJpAmDdqlx10rc_{Fr^
z>BQl+E*!z(-+7uH2^HdA;1UvaE@lm1_~HVA#sFwe^sfeGk+mRRgW3lNZ|ULAHdnAj
zTdT3}GrkwSFGu4FS;gI2>-)b)$g_jZUeh-VV8$BsFB&bUXjwoO%7{@0fP7`#bNMRG
zRPUX9wG&oVM=?M^97eDvoiNe9B3HA1&XW%u9+0QREg3H@pCBl>>ZBQ5KC-^#%f;>;
z%5p$0(U4F=DngKyUo8ZRJzj3fT!zU0Grv)mp8oCxJ0u05vj%KRds)3HSU^Qe{3;nu
zEJ>hLYZ%RMSw2~NA3JHr>n?V+h%xBBhV#1t&&^B?8ngMySm+71WF(<(*{!X>Ou!nM
zf=N)^p5>~(Rsk?A*n5&D(mWb%in`evIczIX2cdI$>qxZXQJ!#2jkLL%Z^w{OyEcRU
zx4<y_1eakyP{x4_NA;$k(Tt+>0v}S;&H_O+2-+izm_?}R#)Y$v&2GkKtA2i9X4(kA
z%nzcyF|nTQc#W7K?i%J_O$v190mY&s!x5X?rj%~pr~7&ZOMyE};?qu{Z<(GP_yGDR
zXs(8LV$Dk&weeGR-rGKz$v3nnm50g>?#Xv*b?%&*l}uN+DM=GQeb%AA(=~3XBvO$P
z;!_T{iHk80`9$zx<1TYFd2O{GDK#ufESt{j=@3F5g95J0Vt;5c8v={EvyZ*<;u-DL
zx6_q9d>B>r`sYo&@vSJ6xIXUv2EYp5pDmUv0{i=RewS@NvVacPfi@uI2&&9I^rPl=
zCZxPsPewiibNbr4gkPMg%-#a03R1wH&nUTNPt@QB;L!G!g75ig7G+^S+QSB-3TmAa
z%`2@#P1L7pnx*h3_+5zrcLr)`7xsw`g^*{Cj-!0_AXKJfur|kH8+g>3;frT0F(irn
zV+AW9`z?O?!T@tsFj|G*DfqZ&v_qX{XHUx~h%xDVzcKlP?z!DQo=P)Mad0B^L>8wS
z0turjqVl`BW+Fx{Q$QXcUlSLth>!(6FNPVU{yN;0gBg;J%6`sUzhY&6GqO)VaNTYG
zm4>evT^97layt21^q8-iJ(?W?u>9or+{I_uc={x_G>S)$oF}Q)tBOQu2If!p^{aRa
zxEUupob5N}$K^cy=EOMz(`fPx6l3+b&P>hkKufc@ijT_Y4|s26H5O{Aq7chI4sDJH
zCFJ-i_XIEpI^)`h_WXKdUfi`Uv|#49#9_s3#=CVi9+w;RkbKMPr2aCNuyuUg9Va7C
zGXhq7S^o0jV8Yn90+SdAN;7wFp2{+~Dkns^kcZRjPt!>(I@npx4*oKtNqeQw(&i3b
z(gcF8;*Kh+M-g_s)(?flqzg5MsA=Y>R#zaPd44J*Kc?8<w$kDww0RK?O3LkmJjkB4
zz_O{)KG&;3YN?nJ;TD#2r)b|jW!S4upq2xa5^o;qxs+!RM}aD#{Cj)?1&1{<r04LS
z89j94{kE6X#8YAChOo^9orQ^y==((1V9veVz6yf%2->o8Z&ai0BPTuGFcScY8!C~M
ztK+TTw(L+H-a9-}>L6aWvVDnx`q0nLPIvOPnMlwWKlPK2+yOFz`-y9s<*qd+vOjEw
zYzm_H;1t#=VBMty5F6s)!K-uxEe-fwA_`<(noR}k=k|Kk_oNjKTWK=DNo(Z$Eq7dk
z=JDhLv@`EZI-ErT==Bb<?6SFOlaNN<p+grmOmhO}=O68Xbx1mA(pJh?CIp>&YERfe
z^1H$>F$Y_&Jl4_(FbndqC=0QVuTn0pse&pvadH~U(J%h~x?ioV9~c{E0Zi8-_3DSt
zC0rO~PQd$&d}X-{^H)Shz$vCAE$j9I!GSE8u<USyR&R$gqds#H!8ktmdsyG<Qv2Bq
zD{lc{470F`Fe+eBI$D6qAQG3>3Z`rY9TBi40%?MWqgMdXsea3!U#6Z=|G@<N!ATSa
z$Xn>Hf&$red3l8@gLS?`^+Zxidy>-t`JED-OTgy<4iVOPznm)J1RSn5wN>7dKJ_{;
ziJoNUu+)m`1F9H^L3>o?8sL$Dc9G)(dK@%YGKY+TOeIotwgXfe0TGKg7{OvGK4~rU
z+F7-&%uCr=F!oFU5T~eXBBQb~PTWfINU#VqX{>THA=4N!vgms{m-o4x>{gkaKzyyd
z%^uy+1g6F)UMi=zsTsyDv#~TNK<1p@JVT3ALHbuIqG%itFPYw!V@3iYJfx<^)kh<0
zx1X9Uv@;v>dghW%C5_LB8pBQ{TGNK|B1=n2&*(5?pn@KBWwC--hz(Hi1p47Cn{cWE
zezlK^;qVIt^UFk-Jw3%mjm@ThW+KwYWCLm!`A^j|wGT&-%|akPkxxtNPtfVhV?k7t
zyi8Pti`r#JW}q2MVbV`+d2d9|*ERiH*xKvgXX&oSI8FqzTA~r&;s5>|^6xnZ3RdO-
zwofaJ#R>pG{7(j4_C^La_J*|P4mQ^3>Q*rsA_(uBI*8XCoaZl+-<!%7M7oOQ6TM6G
zA*%E^YWe^{GuKJmKQ>oD9fp=U)>0xaP$ZLKpeLrLxNj%UdQwIVtE%<}l+Np$XQrpS
zg&dYx?LY&PI_f!K<ZoWTF@LwV(p)U7(gW$`{}9t;rtjfEQqU(vRJNsQQtt5m9meX=
zhQ2=O=zKss8N~CF+2IC)j=rcxt<D#AB|s<r258*f9vtU_bA2=o;)6ew0q>pdLJvA5
zK<MN5$k*2oJK2Xo`^p?|F3d=*kY&W07uEwQTPRN5k5U9?k}RG%+Qu+}6&5$|Fe%js
zA&#QI=w5~Ouu@S8i8fO4YqUs@$3HLQXj+AJT<eBVZQ{}G&Ea<8t4-$JA}5UG*VrHL
z`fb#!8eGZvUzdx@C^P!xH0Wp|S7Vw@-x`+Jn5Ytp_z1-$D+kO;s-@(pIGMa>p*6Hu
zo4esSVS|lMp$YUjRJ|$Qw>fbhNoh2>;^B|5XSG&gBD%V4cr&jT)K!Sp`^2=8Tl}vl
zRW&tJ7rh$o=w;ZDPiTZ*d=4i8Kh{4y<dY=1=-g6PitUVceq}iXB4z1XgNQ>SVzRU5
z&BqSdXBCpIUg>F0VQaXg1HYzNX`t1Ku49~>PLTvcZE`s1W#;_aKxcP!<D^c$t340$
zv=>(QzL{oP?Erx1mD>c*0J?$$Y}tK4AnK+B*&+q;`ej#lZg)8je~$%lUIm1XcXf0$
zRy8Vu<<uQH%pmALT|wEb3V2e=XQi&&=pL$*k;HGd<qum|jD;-YT&Ksj&_9=_5?UOV
z7Yxmbp6TI0U%|+d)6f<hSnOO)LU^l07Rex8#P4q^qO0_c>^`?YX#s9m5X5mLksth)
zFzt#k=Vs?sbVqRhmav@;znbgXRv?hNoWGtMaW{czU|`>E?rCfuedmjLvJ=>X!oKi|
zB)Z)E-FTC5`=OIA3L(Z)w}2nNM&c&Fm!*5D27oxCf2-PIn`$A86Bk-fm`Xd-b*|rv
zi8Z7qsTp{=-UZ?A@EktQ8?rEI0g~VotBRu$<%f`hhQ`3cZo8fCLTkJK3;=Ktt7<?=
z1t1YhyaA`Wv@+*(#5HN{NH&<Vu7Usp-f7xeB`9FF1e+5fvKE1ZU8qsFR!TI?{T$go
z|JSt5a%tlm<9xWRazVmgJDSJ0g*-8)@a5&OOCrp!Eu{e21zjOszaqRcKs{V?=u+VO
zd0)gK%$1%Oyo7$iL`g=t##MnrHr8N_?Cq`wNn~IS)Cvy%mFpg(Z;7@kl0(SyWnEDG
zn6wUpaGPjo{)*BR!u!>a8}%e74`$nOmq_2eZ7B1Yk^CZ&Yc=XBcQ<_>7*dgN40_l%
zt!qi>Jf8AM;$7y2B+RKvxmwA7LXG1)$ArG5L-2hJc?=X%GrAxi`f=NIiF0dLAcd3E
zsNe0r9z^i%wfIbE588Zq@to>v{F1Ri(nwuRdLQ!9?n=x0xN*MMh|O3Xs1g8HsNusk
z0<Hy4hz#MjQ(h@NFWBat;xasczvpeLQdLDzLU)D*>g|l6_0O+<h6fX?tXR$61KH)@
zmx3PnyrI8h2Q<`u&{!d=*p{Bct9e^x7;RRenzAeTd)Y4X(`<O!7}IHYXAd^qrhNbY
z`pXk-1EhJ~(R~Cqkm*-q=BBply-ZF`OLsBJFMeL~n8_nT(!*kNm1KFSpOEFPaBwZ7
z_2*7r*AF^1(<U~+8HUyLPw$XI^uiQNA>;_|5(MyZ{QfU+ThfA<#f<{lP%<4SB@G$c
zK_WQr<HH~$1bL@JIK+LZ)lR3G*G^-;Sv0+=8(I#*=c}A1Qd^Rl&$e5+@Jk88ST@WR
z6{{ELb4{NaqnnhxPj`f#P~BYYkiN)G3W;HgNUjRv0Omc75#ehEUi!(v@D6$Vz1-BY
z3@-=_Ywp}4KczDD5S^mCTw0E!@71+Py+jGlN_efAbA$FlK!tTv<{YtOYx%!-&WZ}m
zqty1Q`3k>yNq|bMUPtxQlmP8O0yV77qn|(GMv}B!^vc*3v=S%;gYAH@5qcz=EsxSc
z_q1H|gaArqNjDRR;RKKzw6MV7c5)3K2NquXl8kN>jt#<*m=VS)1zHLsZKU`eGqiFf
zHN;@Cy>E~{nt)q_#49Kawiwek(^)SgwsLss-?Z8R|FqLh)N0KWQ!MK{iaBllrpVg(
zQ((IPbO4;%i_scA`jLS>zdNXknHNJ;-@gFAtfD=P_?4a=YKw|LaSoSVUrjJfXx|zK
zwrVpsVHsR>7Xv4Y7tmWM2SC<Nh#?t5yc1Y7(>%X+y|U1yO9JDH<e@iVJ-nnt`@t68
zlJfElQ%d_m3VA&Sq%_2Bg9$Y8!w}-0%Mv^nWH{bos0CHTe5+3k7c_IuqfZnTXNych
z{cOxsLJvq`kahbQkbR>?bWOe>Mqu*fwoo(*>k=y$2eo}K#QWt(@j-o*#@@~LM@G&&
z_<z3d{(GpH<Nqd9toiwO;@?8W4D^3c@gG$D2NnN8#eY!oA5{DY75_oSe^BusRQv}O
z|3SrnQ1KsB{C^%5{~pNt-vq*l{3}%adm#J&K*cMcDZwaSIiLBan#!9gZYLp;B5}Yn
z0qMF>B+^R1wuYBT{2ZmQIdh|*kverh=dK^|ZWx&wa@ZPH&L=JH%t@>>mYVOh_C1_U
zZn27)CwNu`cNL)DUS9V2ot7P+zhw*1u{kw2(AV3wRcK|C-ge$FWg3Vi?kXT2n%r8M
z*cs(`f`_QIH5_zKu{LvQDW{F=`9>J})ee9$SfQ|l<|qCnX^Q_sks|5(#vzGs>FW)?
z;RV(MF^fd?`n-4GerPXrh-D_g63#^K!(=n@E9q<@^(+WQY=_-rwO`u@drNXGPQmWp
z66^ku#)~18S7h(HB-B(YE&BH8Lt`<<bxU4|V<(^dbntNk-KNYT*Sk+$Jps=>@7W^_
z!A^uc1h~vzQHcs%F8`iYxQoVWl}Ab9T=0#~gaod#o<Ud-@M*S^%Hu}fiFsp|bgi7!
z9V-pJSi5P+wbt6;^ag>x%JRMz$Q6dg8QKO3q>INAI!e&A^Ldf7V(rfC`TT;gx4w}x
z!D>rGGZ#uzh8Sx?opZ@La;Gqg0-tAxC-W_W_`B5AR!hloI`|$ld0auz@%3o5ES&6e
zImG5tBFr6rWfgw^pbz;)ZK{@#Ox)dI)S(g~TuIWsnMK9W3n0k)L$&X(qY+o0;S{X!
zOG<Ve$RXyh4D2M-X7TPBDb7~r_*y2BQ90_q&-ehe4LpI*m6rn$_I_n@+^v(S&3Dl@
zQ&zf7M3B<@Y&!|QlxR)ZUei$+vi_p(y(FDX2CmB;u3*D&v8z2p6|mYUa#AEV7Qh#-
zZKB9#*H%h?$&m!4hMlatq15Tc?A}4NSL;E2r7<rc@n0?;V{YXnX-S8FO5-_0A)(~T
z_4ggVW@b#2b&N<d7-`?kc<u#p4nyoktDhehMIILE+>u^fA3=R+VbaFoJzPOQgla!{
z7UR${b-)2gFST<uY~Ev{TMUvIRE-S?R-B!m_+CXh!y&mzC&seL`wto=j#LB$68JbV
z&D@F$%5h6vIc_<sUY8i$sJ;W8?%_X?n{dA4n?QaQdc<GNA<ZoZWbPV4TS=kWXCm4&
z$kc{+rI9XBSMMTshsXkfPnqAM<)(yks>oQXuGgafT+!qE`&`-YzDO-a!2S5SHuX<M
z@qhHie-y>z)&cYgoexw*IR#_zk#;^s<*~hzwUqRX!DQh#mn)=s%I?=w6V)bXU5vxL
za1*tAek_3yjn-q5Ksqr@66(-eSOtYindx?zOeB`v%TQ%_>&$+l6eXZ?L8wDgE=GQp
z5$E-Y)={NnT4m%Z9^sxYDDM93{4(fD)s-Q(KbiXc1-$*Y$VkbZMU;?QEw_V9;hzTa
zpZFfOk0zJ9CRn~pUHYt`TF8krykkO3U}-)N!RJuCC^~ei{G^)~P3edF)to6BKcJ{9
z%42a|nNIml+K<&ZNr=0jJA{4K!)b+86Jy6!Ppb$=b)9O>oxYy2_;mUF9PaP6`?pQ;
z-)i^YnBw2n|F=x>-;ePB15^CF)d}Ys-edpD6#u=|f29sj{?2y?*!IZDzi~r#vyAb1
zrd*)3_g9p|=HD)?%J25z*_TD7GWxosM0>rwjk%UXfTM=&c|V$s2R~7wGxW|~zrdmP
z!g-M%OEb<2OXwfJPQe`NSht$Qz<dI9>HhMAx~02XFgcj@y$aTh{Y((ktX@1zx@l9M
zx|K=yn!CQzYocslUsza2pb55)qnZMKP!dnEv~<KH&dCKzrYi=;42rj7n@ZRXk?!eN
zozLV2g-2Y|Co8_xcvDkuNx2OzNMciVl7>+-JIz>X+<`BDi{zG`dt{4B;YE9?J%c+^
z6VH?p%T{+Xk5iXWMt9Vg2}fzkvx+D${FSd=!;W#K$zfA;6mK$H)u0>5V%(s|%+@_y
z<(!9}yRy{p=hWuQnAB~3YrXOz8j%TXS_MA0n)a6}W(ZaQR;Q_I7skxkfEV9pI&e(1
zhbcl-pai$4C!I)~lP_mTR4GqfS@-O{|1v1WflC^lD`Z`%nERwFu~9JZCGny0Wv4sQ
z9fnk`#BM@y<>M@kLeT?X;#O9|p<lt?G1UCZCFc0Mo=*a4Cbb@xB-fY%U_P4ul>^1T
zB}Y~fwozGBTwhqPPl~`~GP0UC@ZWpD?=#`wBE_F)!XHxnhZO%I#eYcgA5#2>6#pT`
ze@O8kQv8P${~^VHNbw(1{QrU!{~mBvToTrCev)Fee@2Qw2VDP&6t7vYi6D4gs)BgJ
zMRw^tFBV%P_E`BdyD<Xk3)5B!o5qt;=pa3Aadi|L$|vfJHv?fK67YX!2Bu|kI5-=R
z*u;&hYYc~#t$wpA(9!XVAgi|8g#x2;F>v^dAO2NgYQMJHTB)ef58?0k9@lEB>+3Ml
zrYmJjy`yek;SALu*eiZ$&u2$|vOlDi4()Nn;*tPCD^t~x+N>RVm7tye3@z5(78K_Y
z{dlVC?*q3tiB5Nf107Z;H=;xE9&=?Y$ixx|O(&vIC1xyC-Y`MNjL5zd#vEppnm@li
z!5GHG!4Uk7m?7RhKTRBiFic=Vy+(Pj+dYY;7s#!tS_^BD7^B_e&D^?Som9tvFf!#d
zCu0KMeXsCgu~!1Ry66gh{M*dtT3x6AD6O>e3QF**yRqK9_DcS*Y5fCfH8Lh2f!JiV
zuqQFiv|JTO{mvhM1I4hMG%(}I==^>5)~^b$y;kg3lIiWv^n95$6Bo0_b;~a=WD!if
zI^&)~9whTdZhQ>gFVPXubL~{v;CqhoVm5@UA51oIk4NuYvdSDb47gV-%YTXD1EMqn
z>bkE|*ut<t-xae8Rgg%Hl7*>wBge%JPGuZ?KPs)piS9$OiM&?Q(jg_vL|qtJrX+bj
zz(Qzc(IZHCEMLT{wc?VO-rYnz(6@u&6CZkig1GjBJj~fJggsu6Jm!i2aKT4E#e3*}
z)wcosu}NvW<I6`!)k21CI{(CoHv|L8TT{Ng2~t?_-e~4G^HcbBwGu?a1BlfS;XEEl
z*-F1(W`5^Hj!I~5K&~G&3u3B^BUL#qU20iKOhA!KK0a2Q5_u@KSb^XxGckQRHL}K!
zoptk$D?-J@udWxpA%+7Xo)aMnIYIF9(;>%oo~>sCS&)dGWr0cp_>kAai2C|BohP0~
zKO<~WSQ<D3EGcdj|89!UoS>L{<wNx#8qH2%=ZA|!+CgJc2ty;jWqSy#E4K7QejyCM
z(cKLKGsa#)ppYWsP3#JN%zMl#@G1Nfl8T6xkeBbE_(UGRx=K`C+!6&`F|+_135M~d
zu^o$BJIXiGANebIMVZ8r8$&k;+RAJ$kR%*+cfN^r0~*<tQhPZx=4pNqSieNQ$|opp
zo)>{G!z?;&8kO3NoX;b6VpmXNq*y6f@JowDlQ{9?Jc^6_tYq)7Ie?b|Z4yrDZ&9o`
z0^B?gU@_H}=$hM$)<+meu$`E9^&g}dZ|6Tqu>)rAneBExwM6SUso+<r#=wf-q&R}d
zRZGnlifl0QN64<rWPGbopk0)=*sFk5`4^E;P)3;^>%q!!4`ALQT>ywu5N6a<*yMd9
zJ|D5LePEXS*mxxAAJrW4DQfSHEJPO$M6b_b7JoCv`#roi{Zkjx3Q4h~ZX5Xzm+x;F
zA5RyPB|cmuDqa2Q18Bw0Mc<F+`0!<>y<c3PMN^rY!ZJcYEj!*Fd%6j7kYu{F&ZLbJ
znqz#2*{xHYUe;c$tT-C{ecNj#FW`%_oM(`o(tPOo^|cnjaEHZ-P_Prtq(pTOWML+T
z-q{+2)c|?dZ!Fd(Zmg^LdkLY>fE&QnPxigoy>9Q{%sk!i?`vAV=VXbv_)(~4;oXCl
z3sxKrvUhb=odxjfSbIc4{^aM~Vws1(xUI$g6vc<heiG@XGQvl;85678OKg|#-agLH
z*Yl@|ZGHQn(RUM<_JL=batU_acx*A`%iR7i&NeiRK-6V|=x=J`-!moL@v!WUsV<QT
zA~4%!CU_NF`<krNnDCB!?E>l1cE6566s<CvL~e=Wc-YGR2sj-jic-Z|P_}BhKT(Kp
z($GQ0_i~5G@xOGoDUtXe?VWizRofr%%`*`Vl+0zGlX*zyC^99InKI8Zgba}(V~Hg5
z5Q@m0DH3H2nL{OIu8`^7yL-EF&e{9?^8WL_&(pb&d+sgg^L*D{pYvU7@AchlGd>_A
zF<Wt<bQej)nDn(euy0vmu#r>lu0zY}YjMs<KF=F>Y$T2_xiG3dx*^&uS#j>G^p*UA
z=o%*S>72ZGLaI-W?!}c$d1+OF#mw2Ep!<o5;W7~`Z|0@W8nqm0C9x_8hgLorcfZ;%
zp+8b-YpzB4yOPfE+2X{X{&aRU+@8`klAkng{UL_ep;0GY-<;x>tISc}x0rc#{Mk#L
z^!y_fe_$$N$qDZdwUv%$KJ6&e%TGQW?>8k`CX!j)4%8G^XHcEAtqo7rzw#3JtHJ#z
ztm%dhOYwO0mMz$c2+tS>MA%b~^k}fG5ZT7ZlIB-*EX|89;Kmn(4+b#R;715YD;F1R
z_LC}n78{XL(Q(<Qup%wIILc0Nssrz^5osJXt=Vgpm)v^QGRGd1Ml?zai~4M?g-LGF
zzGL+=O=THuRG_NU8p*emh`=k{yWc%}a7?M-dADq#C+qlKr&6brs!fN6+HtZQs%ELN
zUtZ%*k77QA@%&M2D9&8?UZxTo<Mi)XuNNfV72}aTzq9w$3BIUX^nsxzwVlLc&ZD&X
z3gIKf$D7Z@Bp<(AcIuH71x3X~<<F|(hyPR+R~l}wDh7w%|Ea1N7<!$ra2Z?Knp)Xf
zIQ_h(Cr{gvor|rd{aleowsw)<BNx&cU;T_b;<)szEu}^N1eKGwKeA;9HBYjAVy%tE
z%lMFfbbr=(2Zf&R*|~dq)MsP$^hva{HHoPlUvaSIHeTV;#Xa?!+%nq#GVUp|7o2!{
zN&3+V_p@%1ILlC|M>wjrsi?goROV7t<*fU_uFtNXG?Ac_q^F7H8qU;lYe0{C+Xf+E
zv;McLV!pps#eb`c|5g?Mtt$RoRs6TA_-|G5->TxjRmFd+ivLy>|E((iTUGqGs`&qn
zs^Yu~odG$tRmH%GEK*hR@2!dp*gd^QZ~MHs97u9!@4je3LMouDxNibODZ4w+mi&$6
z^8;T*EX*S;2A)RIZd4r#pPiGBX}EMZzx(ArALaEn8-2ZZ@paV}PZP}rf|r_`H%^vc
zDEXiy+Hn4Ip6P_WSsb`maf@?OypICGr4(tt%!E<ZzWjy^&jm^v`R7ghCfLuHe|J59
z^n56B{9uTAuhN~HBDhtWXBmzO<V#mr@vh%(OV;<@^zF9suk}4Bciekx!Bx`fII~m1
z9Szafhb8$7_Y6l@ABnD3*RZ6tXs=;fV=yUXd$0V>RN@(#r>C;v#Z&jA8CHp9d_#q=
zEevZ)O?qarFz9`stUjE)IvHCbJ&pe`u(;TOPE$5utm$xPPOD#R&Z}0rptGr)r5+{R
zLk~@2d2`2Pc{8c6nwc#PF)WrF#LtqaInGsh2pQZa?QD1{{Paz#LsLwvb^*I9W_LZy
z8RCFurJ1MmD)hpG<0ZoVfpwL1T+z8vXC&-w_h6}*zmKwBs#L<)8gta;o9bn3y?c{Q
zPw(ycTgs8t8AJaS6OHv6Dbqe-;jB^|&pnSfZqGf#dgiWmB-Kv1lGn$*^eWk&t?OPX
zjy)|&UIQ^FP9MTP{K)mdtl7*<(I1#N+~y+5Qszx-e9yjSti1N05V+$zDT;?*AEnpx
z`H8*Ch1D;j#t$cYFa)<S?31ne-Y9tQXL^y*M`x1xJ%Ph(>`EoDPq7lC&O%~(EWwsq
z^LmpcF;j+4-9((Ru+h<=;A^4yHNGFe4Achakv6$qzf>>#q>%K<zW2$=FV>|D{M2iB
zFTd@k=vKVod~e+SW=$=ttjI)F@Y+7TRQIv10@<{XTO|D<WEXG=iO*&my>oQv`H-h7
zS`pjVlW~w!DCwZQ_fz(X3Qcu`AF?M1BcHa)wCrg&<q%J$Jo5HXa$)R*$9OzuijAk6
z$?eaQAF<xA6y!C{W@vriyhOhq?s|H!i&}iGQvAx%tQeN*FU**^PmW?ZU}A?@5Z=nG
z9<f-kz+UX@!e)_JXWDAXIWsKJB3*sjd1}#9)8F8@zd3D5wq!}I&fTzQ6Q*s;z9$tD
zc{k2?V;?xNX7P?9$bm5=>>Z!#XjPkXA6^3w-qN-k@WE?JA4hrEm(dm#|MOMcUQ}#g
zcbym5rWh>mS0!J>T;u2tAmL-VOV5han>&52P_;F4mZe6l*)60HtFc)|*FpS75XJTV
znNLnRe-n;LFVW0oH?y*Cdi5xbkm>jl>*u<AcvYtql^tX>9G>$nJk)Hg=-@uo!k&2B
zTxXh4qdjV__Q5d{>`Zrt_v}}{W=T?L9$uUnc0Fg__Bw#%dMlo7z}5Q~*F(0PmTgDE
z-o3xL)cSSvwm}Xj8()CbL{bdijnS{~1T3yihc$(5GI|G0>O6e<UObxlo`_4Rg;slR
zdC{9m50f}{f_C9{$*}c{XFFnXQ=;z8Iw}qFI2Wi+D07gOvfA3Y?X0K-jvD%XiuryX
zy8o-1;{OQ$kJJ>`P$~_)LR(Y(U({etasR(H#cPss@0ZvYS1hdO@*ZkO5vs`yV_+S#
z+4pVuGsCr9T{dE(EXR;$y!QT0W@(Wkp_TP}8&^g7@B2`%tu_{^JX%m4ddgs^V?fiT
zr}$3VsaeF};j$KqO@;k5?a9iK)l5PUnt>nB3=|vQk>@JLs7_mdewA!ocMmg<hv{=u
zv%hiyKl_iv(*`$YtW@P#@87(0Qcmt-Yt^%JUy0pSC^Y6r@afptU%J1F%-ZCt%=%(B
zVsOH<l4q?b*N5jeg#w4&=O6J0v%6kD<*{n8(_DM{!HQ;D%BpJdv3P*XHQ_Afaj%n$
zL%B|QIT@O&O2<CLo_07;QSgvMuTV#_xC@`^LN%^TR-ueZ`~@anYpk^Lg+SX94t>m`
zNIHeyk614?S2H<&m~GMJJ9c4+O5E+`ezGU4S|gM#_C=?q+1L>?>t#b`Kc3hn^_Oh1
zc<G({zgQ~kvVKg{Jja@U>$M)2{JGFTM@eM~KMjYx@W&h9o7KmC_XfvqR-UalVRInL
z7_;K&%I=J9XipC0FNu60xRoyG#%Xr7Ui4;~VZ8B5z=m49eQAIP=M1w7OPt_g`Da{#
z`kWt&ontsV&l`0PZS-Bbl(WB;np0}qPQ_sU_}|Kk`Tmv_|J|wh?@q;kcPjq7Q}N%O
zivR9Z{CB6~zdIHG-KqHRPQ`zBD*n4u@t^Ee44!a>MoY)b0%gV2uw})3=*o%@(c4@X
z@*u)8R{E!`IHyr<?AqLy=x90hQ0@q#Lll>4Cqt9KvSM@Bazn-#x6+Vv=iHW><fSCn
z_OcXQnV`YD;$}h&lohWQvzE_T>mN_Bxr9?!JFl!?3T#wtq)*66y?_3K`Da@^psZMR
zmWZdU&)H$3;8k4pihdchKb=Ld!7I_iQ|=G?GFLI~+SwXS(kiU43e=4TFMPk=w?y`{
ztawX$9oVQCd!~(?yNX7sgt43=;k8p@m*!Y(R#$qwQ7Hd)r&i|IX2vAN^2W?AKv^+0
zLk>_@Y^U{E(lZ7uD=wBS=WJ8pbzWY7TW=AlB83x4UD)^RHYYdUN7LmI1_F-L8E%St
zuUcnEBHt*GyV^_-nDh1=cOp*vp2j7%#jQOOs3p350>>&OPB*NvgwR~>KoduG(VLQ)
zyMvdyyB=G+U@@Qm*mUN(;ZP8<R7S#Q`NelWsuisR?@j0KtHir`@!fuYBc8WQ^G)?7
zujM%M+nG*uHiwhCuP|N@o|?b!;Z}QX7TBn`Wa&tE*pFLpOI@_6eVb0FW<U0y6mP~L
zKPmEou_czdK2u2Vrjb5NzRlHE%*~~OyqEn<wqak@dH0>G9&t8L>iaQEtY5)+K`ox-
z5p95%qnWn%;#_c_7r{I~-pl4o1{foZV;9$toNUwy95XoCp<{R;@#6C1^-Ew;F~@uZ
z;Z{{a{ar6pb-GsfBmw5essa;F%3BdZgQeK=SK?Q%ED^EJ9D2m?Cdn%N<f4sf!^JnB
zz^dZn9-yjN6-O6u_LBB{b|qa#<_8I?^gN2uEwOx=IaG?|uL#1}TN{r!m(33_cq#5H
zQ)E*k`yk8zPFC`L!=&dsS;2XRtE_Ydcng<K23Vft#XVqg(n}-Y&fQeYHQfZD9!_qi
zs^!6j&?;8X8LZF&ql(ft?KHKxXd0^-Z{y1I?`Z`mf0h-;GK92xsJqLGb~~G3ViDz}
z)IC=rZj-D?h}Xl8P^h>~Ev0?U{`(mcGd=p;mAem!byiPwH4X3=SUk<gE^PYtPg$|#
zA@O_RwFG>DFD!*z-iz}z_0=+ZifT%mFjDlHh!H2c(bt~)RKaC>nYag^o`e0wKvzs%
za<8RQSx{<G=iMQyna0DSe8CLiIb+dM{fC2M<!{)F@P9O5PFwzLKA%I4W1@6Wj!A$%
zf5I8hyEb{h^fX^R*4*fYN5PKDf(mycygPJXeC=(z)j(Ob>|<p@pXK-fJB)UZzK$90
znR4fJf;Qm!xj<#HTPsZ!8`qcvLxTBLolV<2d;1h+ESrZMa@PY(asu#_ilv*_R`Qej
z7RG9Wg-^3G3a2{WZ0Jg*nGB9soYiF_(k*9a=zJidKC0{U^vWsmucua4Ph4cm)Jmw#
zOTnM6B8?j+F65YsuCnQz;~8y~+E`Ut|7^%Jve}(hdc?Rlvs5rVZ{5UhYkh6;%^5rH
zYlOxEi4}6@((k`LWx~Sa!BACvHvjZQiE1SS;gP+SvYE4VbnJtdop425%yh`ftHeH2
zvQ8&%jrsOeR1gcPt(^+VnY`SPjJMU)Z$0zO=|T|oShwo4$1_z^ABi{*)6_AXICLrg
z;^^pYV6)<yFdoC|^_lnWh1_YC((#{V_GDnT)CQk9IyG!#<JRQm^6|<`^<$wT9@cz&
z_C4lxshkh%3SfFpL5%tIQ`I4B?}a|PLPLX^@~=Yg3k08Ue!H$N&r*+Hm_sPh7kREH
zlDwwD?Ev+glPSC<OU6Gwab_OKp$N`@Z?e~v?ZIf6h<pHt!AETolV+i$(fNjxO9rtN
z+E?<D9+wB)fBNo-A~p7T(}jVHF2nuRCKX+#w527X?R*k)rx{#>=dzVKk2C1qyIC8<
zj@5x}?yWs!I1^KR$|OQ?p#1Co`A6a(zpy!$F1d&G1eVn@PGhf)y-KC6yrO!KObp9N
z;+unhX1O-j(j|h2jsrfJdyh^DJX}m5FJa;2H6QCVx~FkH%5{BgG0N%sJdWR6p6OQG
zuY~eON<a2co*P=3!_?)iB0TZns^)p!mUb0j!(vY|72a=>Z^M`!NuqJ~Mhk@Ou~q4y
zR@%ajz9YrvTh{N-93m3M_?3y0xzn5a#*yoZPGW1>4ii!a%DF51hgj?P8DBPeZ@e}h
zM0-7Pi7-YX{LogMtcJYML=Z!UAmypTcjnLgA88%<E?%YT6_RnL$)<rq3QOS<;cX4v
zQMw=|zPirkCwUbSOkRwKS0{*~ySUaH2*zXi7xo?IdV2M`sxjT+5EFZ6NzC;dY8Yqi
zC)*RGbBU{P6*jZ6{hxQZ9uAje;avE+Y4MFLgZrMWzI57w_$TB|^~P$BN#Fb6)J}DF
zv+)+Wsfz5YgMoB8ddD(T3}*%p=A`*LZK|zg`%RqjROkG*e@roes{TRc9Sj8?ziv8-
zfiN0xs>`plVvZcYyzVEOQ3pKN_~l}x+xg>j<FCsY8@FbIw-5?(|F`;LzW<-<i*<mh
z#r6e_L%{zaK6ugH)!Z1xN`?XO(T2X9jyhmX;5!lU3<czmavQmrS~>r$v3?al3Ouur
zym@qkgP>h9#i@HH-}v4sY@E=L^YM6BileA66TPOM*peiYU$2YBH1}i}iW~91y3zGH
z^QaQPN7f|Mt^4Wnd2e%w+r#$q%-^9~HKptq!ej4h!mtX8z^ADv&}}DwSMY91;*#p4
ze43+QS{C$QicByLv|h1xDh<t_GKdyh5Lm2WbQVjd7(8c`bu__KEWPJ>ypN*C<hk#X
z?e{wincDs31o{~7h<5cb$YW=l2=W=xh|%}Bl{ZI6(m(2YU|*dg5Ov(u`tws$TT|Y|
zpm6)qv$SiY?}P;G1F3vR^5R!;%I?LKImmnI=u$Nu)GH%i3&L$pIdIf&UsRKvWd_NU
zXchWpTwDzbH-~%TvE77IOuEYN0_z3?%@(np*y1KlW@FV`<{op1Z63K9+Sth@jAhwJ
zyQG|6!5#C4rZ8l(Friq_ge*$GIU>0wgPGo1=K10Jb^1w_y`h6@;#5T)_lWyVT`Vfs
zW4RN{Fi9g`U#S<)Vs4yOZ)W}y<9K>#)oc1_pklLj(yE;<)k6>6ftYXF(NChHzD|us
zxiD1>yp?_UDf4<!G<$+l9I;Ydp3<Hk_u>KewT;o<(h|Ghi(mmj$M?D8t|S2+ZwE|C
zX;E}M;E=PUo%KKU^(C>7ZHIVCf|fmr8h{63*!Xjgh<&37JvsKtGEpL7FYAwJ*;{Uf
z=>xAHWDnD4AHZb3)^b04)JsHD%SIy4{lMqUAIamnSJ|!<NH@lAI7=Nn+t`};Xj0GE
z9&d1ezR=k-fpo1$f*9!dKXH%AKfIkf*8YU_j8Y1}vC^}Vt+;WAn0kLFg+~@98!sx<
z-r;wBl_;LV4LP%tL@tR{GJH3=r~k-}M0&g1YPb@I@G^++a{8%}<C|`ri?3(Ab8)Ja
z*)q!iu1V^#*<PM?=HyDpP+3d2vdKy%b0hPF=X7<ei`<9XXD_Q+vPFL+)e$9PST$1L
z+`?1kF$jArzCL)@qH1<p+4ytt*xARcL4AeiJBMTl@a`+k^{(QD@>M4!WQe73DHNVO
zT<zw$cd``!G;yBMVXfNyhDRUeo&b-;{feEdPr85`aOp}#hEan-QAd?Uu#D$m3g%>3
zmvkPhRVp5B_qyKl8TG_tU6QL}p7w37a}3|bob5Irgm*o{WO0`XIe|U))F<&H?>$ZY
z8xFa#EAktw4@`Tm@GFq*DGWCJ-L2qRhNz1l`3s;sC4jCyf}&f^+|5i}oUQC^e}1RK
zBkR-}fPS45e;zBqG-FvA&s6Almg;b^{1dhJ!g7@3!w$kJwB@?b*1cWFd9a=I{RfSC
z&jdfFp4)@hd)F{bHL-uDf@$VESG0TrCH3)$_L70PMm(O8tGCKLLqhbibd?J}2`mJg
zwM|i*A5xmHfA%_GQo&(-)qnhZuOq`rpZ)?D3FepexVg)tA8jonR4p#Lkg;8|kNuWe
zBNg8LX+fM+H7`J&h%(*xC5?^gM(BEr!64~|FiE$w_E86|=up<H%PLKCmD`@+i<l1g
zqc{dm&ZcILj@SreL+Nxxfe-N8<3N22#HWB2(LC*!Z7NFnXNxp=JJioA=PRmmbG5X!
zV=u-fZFpmQ>C5WJ`R+9f5y{ATF5P)K<J7Gr5k+3YMl8PQC6y&*jhoUzJ-v=v4dH_6
zHBs_CC%W!XF}*u_H;u;XhH*OKdHHdH&RC+OsaFTXevAlZRXpX&woPRrjTYf5B73t)
zwk=`;psy(<=k09?Ffi_uU|{S+Y^ST0nH#sgt;LMCm1gwk{hK|49NnC8oD^ci=P1lv
z1$!K`ZI7s*5x&Kgs!elV<cb=r;8oF#-f+%+&+&Q=^YY=J_WK%dMI@3r9_uuIiy)NM
zLvqcf@BFfY)N8%`%HH9vn&siqk@k@}pX3rdk(B%s1*4@$Fnq&>vMJRt=+9tblrOzp
z93y=BK!*@R^Yz;#VORE0{x^r*M$VH=w7s4x4YX^MuA}HFTH>B;Y?|=2OM3f0Px$6r
z`*oJ?#g)Pqw?owN;V-;Yd+P91n*|82k9X7HUwEUg>{F9Cm3XCK#o(0T;WqON(`n3Z
z#Y-~>GM{V6jFR_Jxu$d-lyarv#`g-&nUar;3(&jJM{5&}-9$u0^!>TM#oW^{Oy7LB
zUTfURI8xu^iC!3H5@e4)u_~!W;;<gZos3oVts<gd#->#~KH;d(;pXNRwP>%cq0v=V
zv8P!e`dNMWY+(IMExb(`nM&4#j)8#z!M69RR&U={8uzZ?^toY(OvZZ8Ce{@nV_`Yi
z1<bnougQPYB>n!woRO5?UE*9b{;heTtrOM;9e%5;tGT^{3Aq?F4vD#U-aR4pXn8fd
z<+QOD_-UlS<+U-R`T^d@*t8oLip^5B*M`^P1LyWoX|=lz4vn&`Y2=k+Gc|X9zo7WK
z=-TGW0@(l?r)y<7zqG>x4*3QCR~EV|-E*@u0biCoct_`FG>e4g?@<seescEq-bg<!
z$LkQWUX*epTu>;;((2O$d${ov48l|nwQ3Us1wYUJmEh)1r?e1Wm752-_D<iB8{-=0
z2%6(}CDZkZOR`ZG!^vW)9apWrcDmr(!!<0tAEM30lM<rB%=@TVt}m2d?Onv_bQfsk
zxE%bcC^dPQH?)UN>>TO$%FWH$%^Q(rANKmji25^@@Aohhe!p+O(1QDN|1^e^6!Vkn
zw6rhQ_j2{5QrtYtj$&vz7AW1>gE{g^;Cf#+#%%93?@t|PJex1bu7~^XTTXvA!pAJN
z&%S)f!R3;)1@rYjjKGuo$liPtD=)L;tCD?eW4&HmG<u#uHhaj*nwXcr-lyY9!L^G0
z1QMTSX1FwJ$VL@>&hn?OOYnIsxDF(|qr9Qxrb6<SGybRw{YZc@%_|<_w{HhT@gvC8
ze~93#P2fMCs5t0D-agi+fX%8<nSOSCEq3FT*qrfe2eyGcCIj7rV<uldOzRzCn3|fM
zHGizTr`kB#!i-eBy=AC3WbiAcw%_<<-Fyk=`6aRbj<&Wohq81vajD2tjIA$UevnHQ
z<85_cI8~XMY3@(M>2p`)gWvO9nLWXg;`Svg?9}y>DTxF{Vgo@Q9v%reM&3-%@^I|#
zJLnDtW(jGT(cpXd>G2Q5Ca0v%On%W<)}`w0=~2wknDFsu;PP^qG+Hc`JrMsPOC#%s
z;l;wZ7Snu@?{cfFh0NEVl0LT8QfG*YiaL1uinPa^R~kd)UQy+&?ijB0iRFhb!yNu+
z27QjWd(Vt3TpcoX7%p@xeSMU&I{pgv(Zjx$!42R1yT1fAB&<CZCpPAaDD9^eCgl0b
zT7A~jDQ#HRJKNz>q$a2Dl&5w8NwvXKvjZ%X#>U2}#l^+CoZpYO=>-M`o<Cn*o?B)V
zt6&x!?y#>nmxh{pHI;{|Y~OtfXDUX0qSCuTKboGm6y%(DE`73oQ#nKQISC6*UOlHc
z-@&jn6DLE%s`>sy)@NPB?$Z(P#W$@nR6b)r)0E0?giX5~x_P-mWA;kf<b){pb%~e_
zYga3)#xG-uU0oW&;nYI$s@iY(Ry(@J1M1owsCts4u<Q7{usJq7Ihek@GT+L#v34w2
z=zNuUA}=?$mYKpSO?Bi{!oh&5&rcQV?%PT`%XpER3MUCkNZczME=#MX<&2c?4X3^)
z&d0~saM4IhE1~lu-nU}4OeV?)3bzD1yGX>cvy8usz7}}0X~8aIY4UkA{_~w5OXohD
z3hLt##rv1JQ-(Uf@6QrF*vTpHJkoAwIvkXdk--@6oEjx1EX+zpq*F4zx2v(xkA(3G
zm7r8fP=cH{HTQIGL^$(a9hxKb`ot1PglTKUKbe1)>ejhO)PIhvjc;Qze)Nso^s~FB
zLp;wuOEN0`z@;_2UG%oO`JF+e$BTuR%(UHSALQkJ=V>tgveFZ49kG=o<VO=7qukLP
zuUa#F#I%WMHitkdko<gP>Z%!L&Y5Sr#;w?%bhP51O>Z{53!0Ah&|+P|=1<tW=-^>g
z@~OsBrmAdsA5~hz!vpmF4>|Ai*~(9c<7ipBxqLbK?WExk<Ige2&IIW<Gcqz+th^Vv
z%^j1YIbNh9!RkRy8s1V6&G%@4Mds5*oYebejDx9xnRnX!mGiT+-6t1KpIZ*d9lPM;
z(cvfIIl@KR#WVB4zKDeTjH@wrg5XhsilyhDxGFnam)7Zdd3swpk}Ya3;Y^>vS>$C2
zoPAZA5?J_R|2%hFNR`*E`eCn{d-*ncdUu_zLfl^8l~40?Sv36d)W?0jeZ8v6id@;_
z+Irs5kUk!h^ZUFo<@E}J(!mi9sS|EVU&u=66;q-lipRwKnSEN!DYGh6nATDk7#NB>
z175Wjyu`z^m4BgS^2#C7SK<&gb;6;Ew++jl#kuV4?04Fpzgf9{jlRk_OF6h{{gOkf
zxqXGD>z)|tqfTs@Ebhs2SHDR{-u@`n<v>9}L9SnxR_wG!clX9qw<juUeIw4_A~?Li
zTkBd31oRb`Ru^}_s;k3j&mf7rFq}9}<a2YY)T=H2X5+XocT=Kp1W`F(M2yGAt?TD~
zQtE8j7J71DF1LF-Zq3t1w|OWtw}fXV6UtA&*s$@$Ba3Xl8S6L3+{6&UkSw;kx|Dy~
z!}s%4s^&sHhI-a1npD>HmAI)bvaK|-Yxh@g#T$`{WOdEPF%W4#vw7HYY^vG2V3Y9o
zWOTjsxWQX1va%FlV!{SYM!_kSlbwsBiJ76Potv$-ossFkr=)>ul|=Q@<X^Bi8w4@A
zNOCy4lsNMR{bxMhijO%S;`UrvmY^iW#0@sH&~>%ge?$Ed>vi9Q2fBJ9H1RwNU&Pfq
zFlSw^FUjJ?3oT`OxWeQ(cldnB{GMYC>?=b*5^m3r*}bo}&s64ZmODg$ub?Z#yJg?Z
z@>@-*#wRJ&0@S6!w2t311I+x={9Y+qy5AyElZdH^Ym!=*Rx6I1*GetFdGU9fw1G+2
z7jmsH*+5&=u`n?90PilgR?b}Zz_Z-{nN{ZB;v=i)C6QZBVq_vY^;Q@Mo8Ykm18EOq
zQQkm)<BXLI77J|`CQp3Q@_i;1QwN>YNj-A6G~P!eFT}sF%DCU{t~AIbJaoSPf=yWG
z*R#(~SZcE!<=hk8J0T$y!6ZPgeHhmuTjT2nl^OX#lNmAKe+V+NG|zcui;L1J4cy3k
zeYc^UpvB?FlqWrYKdA=r__ac{nN5x9FY0sOs=|(^ls4(LD%0+n#@*Q{pdJ5JL?Si{
znDl~EWlSt8Y%1W_2r%599yhT28jxB6^y)!ivV0Ky^uMokKkz?xUVq@@kyAVM^K@f7
z#TB4PCOOiD0l%LG{D2o9e!bHD<^eV$<(#IBh9(jRjOIAs@|!T=6VbRB7_7e_z<avi
z6QGw6IVMI<W?a0;TVEqa2ZDf)|9mn2`wqUP`vu{^QvX$9I}eP~hHvBp4sg7d@cwyJ
z<u7Eq-wPPtwq0frfrwYLegjE?vjV7QBi=i|oeFjeNcq)l@M6-hSGwOM3hK5?OWQ#%
z0Ep>@h+`{ob-|hPpDPT%L3D&j;c@+w8^AjUxa~c*qu2pdxgbNQ6$Nl^CW?W<@QVYy
zr~9!0e~Qr3UnpZElPhMnrU*DNGL=&YCV7B19tDm&{SMqKpravh)<!P2CYC5@6K<?_
zTY!DiK;ItSK@(+xrUCiS5IVvk?K9&W0Fw$ZdS`^*#~_*ZcGgxV9w;J={hGFPh8zQf
zj{}ib#s*1qbg{KHb3~wl9{c||yV(J>Qw@j@yu|S9mF_18bPvMO<KB6}VSC3S$_9=Q
zzh3En>SD;Oe^Tsr1{hENsgoCEfe&f{KRX%0s2M;;+s?oNvih7F#a83LeYi89Z6Y!F
zQRXvjfI@PUmOm7T0y&?pNFcU`p9gh513BDJZg0c@-w{UOn*(&vf3I}E3Q1(nfBDSZ
z8R#!}D;pyVGw!R_R>s>K31aoVu0H$*Xulu$&g@K+v5GKQgnM}qsl75jqo05f?*WHz
zr~d6K@YLV8B_J}v7XQjYz&G;0SGwQ&X;>zT)C;!Uhb@Y23P?qPK+D<xqk;KDQWctq
zyrc5=I6-a78$XeK0MhvcA7V<k1#H~;Rp96O)lm>B9SjsYrN`+*1L0CSNCjK`>u~N&
z>DIvCA%?lDl%9nQ`jv<OG^OX8A;t6y1)VUVQ@WNLByJZez1{<ww%e4B>jlX~nbNU<
zF5RzWl2!?%bbe06lr9Zi#6;-QUxyE6N(Vibs%<W72P8pG=^~LxG?<i*DBI3-eCif5
z3pu5O@hl~|mni`8oCAJ#GOnU#Af$9MfZ_i<I}Q&Rl35QRPEo%hF$7SibnqJMF{jzQ
zB_N7ffV6Xp$`^~+8UcuuP6%+IQ@VZ}GUr#OFoH|zAa<&Q{lfyvlrD511`CtYL8_t*
zr};;q(a0&?CKaBFI;DdL62SeJgLbC$J87^?6e%5y5jv$0!0_NxI>^1iTjV|hr1Wv5
zluq;zI!=g`4g#T5I$JR`5H6*IRA`?to<LE7JG;>Nw6}&T8MRLX)i78zK1JFob55c3
ziJ}gkirOde06_aBQ4h=fW1p_U@X-6D_=5e>B;eC9l23&%q2t_@Pd%N`KscX>V13H#
zf}--n`vhWb7v?R#VxgLH1`fesVSECqoS{>hsC{}i3QygQPoKwNnJ9b$V}#C6T<@TH
z@IHZDJonjRaLPG`<dXp~3*R~L3;a9f<lkOYx_u4{0-^I$z&tb%oloL4I06l%7#J+T
z=)3b2=E4FLRRFzDCK}0-D!@=&v={Ld#%v1)i^iuSd*TD&DNG8|Da>~)Oo(F&1Pf46
z`ve|V=={Wl1IvW)$(gI*h9Gc2y@C1A&QlmO7#@0`3Y;(BWd%-+WCRgUVct<d#|e?2
z;C)i2h6cj<L<}6xQF8}{gCrOjt-wO}PM;WPps0fIK7m*}w&PQ?Kz^D6u6ph~CH6fE
zgN5-4r0&7e)0_qbxR2!16W~%f>hCBJ>@Iy8XNP6}u}{n#&^&mbz_y#bb7vX@@{<=*
zeo7IB=I+X;Iw@!%I-j<Gop$<kM;eNH9KBET&OdHn1AO`hsJhc9O5ofIb<YD>G(M?p
zEItOuJ7k~w&%#qt=O?hApz{-s1}qbWPatpm0o|FO<Y9Q|eX6VsR{IKE1--zDn4jtm
zq2q+`2@Di^yjwPd2BPx`dc1pW4n-A0?^Awqi{Lqw<6XZq3>J+~2RiS$q8{%Y+~KLa
z@hRB@mifm%jluBXeIfz6zwb!ZoH8)p2_ubn`oYj~?#ic+5zs(5pTIDo$Ge(HDC!A#
zpFpf$Zbr)&l;a&OPz;JXKLJ=6pFnCEzSdjNC*=He2KXa5Dr%oVCUk!C0;Xv&OcXwW
zJm~yX1H(h_)046U!*yW1b3z*LL<^wfgzyOr6goe-0S^#`YWVJbQY(d`3ZwUF<ZQ6T
z6iT1+>tV2Hd^+F|%7EIZ!RPSQ-S|Y%49omupOj&E=zV%TB71fX@W~a)r=}k0ICtd}
z)(|uh&L>j9Qp7FI!LKX$O80vUNfm+j3B-O#$Wq|~#yeW1^Sji~P{9IN7@t6D3jaGE
z@Oma#?|$@WnE(7M-EZF<JasodNdv_TsK2H^BSqm87$f3-Jio$B_w$G0!TSVqOMiUw
zJO+$+q)6l460pDw6(@vGAP_n~UBJbH1n%A^emn>&`1D)2>zQD%S00zV=RxVy8Wju{
zjZehAC-dK<^vV7pJQcN1V5DGw1BvwKi}Bxg@Gadhff|<i$36|h@X-52>X1)-6!3`<
z$*0rDpyS+?PsQBOKscYk)gS2m6be*+pwu4R@eR%=5S!?GBM-Gt=OkgUFg}6Q69PW_
zQRk-@GVs*h__TBqmWjfrpDh>M5@iFfKV<<I^md-#9hZgX!TSVqeP1qWumL`iAo-Me
z7Mcs;69|OPPvpALKscWem#=m%p?^d{-PIC0<9Vd;el6Sr0~|sj4t!M<s}Uq>H%sUp
zj*z%rETK!dK+|@+gzoDG$=t;fdZh;>4doI#sGaLdBxVbcuO)yv#?Gp!C!t6*m?d;X
z*>)~}m)}HYAupkW@gUyL^lL~-_nSk_Kv+TtZzhw^$;SKwjGmf6Nz~4&s0U$)aR{PZ
zLI-chYM_|HN4<pJa0{7&x+)6fKreaDMj><lrzLa{d*c4Nt-U}awI~rU*}X`F!NR0;
zkb3SW=8F~Jm;)zQ2vbh_`|woMDIH{jU5I*$pq~QEM3K@#UW>5dC{R^|u@8um0fhhi
zO7}}f&$F`KzK{gsR4R-Mo&YFba-eeU7Yle#_Y--5HarCTKpxBrS{gJDe#8N}ZMW49
zuL5USy#k0=6b<sAxey}`2!tMSZWloV;YJ)?fC{a<Z7~!T%?t*_zP}Rsi4>*oiq$Y!
z7~MhY1Wr5VD$v)+x+m7cQ+K0#e;q6nh3;UC(7Myqqvh??9mI*K(I=qRy#zfDL3fY`
zt$R)*ba?Q(gWT0qv=s}0?wv@wzior&Lg)?xp>-$jfd=kg_i;!nngRqc*jLqM2MJK>
zUNr(0EPzF$``)f^`lxj$8i%K%)*Xx#%$TUNyTSx4^N)28LC-_h{rIO6qhgfVop};%
zcz>k(fe+9;^tu}wtJQu5bnijZT>)6=-Ff0Ch^{^YUUx5`m>LQQr~7%Jx1raI&V7fX
zqN$GnvG>0dxtjvIiv!E#JJ%l#fLl*c2m4PYgPE(Dt@A&-Jy>ra*+Hs8SR}>|K+kfZ
z|94(L!w0@waMa!CE=LH<M4>wvV->N^2PYs#VD%PZG82rRw^Mfzci_DpQyYNV2V96m
zn9LBN4G%$gkO#f|AV&htgV!D8j?D=giUFt3YC?#UnL27{E<|<*fza7qf&m(cPIu_#
zEGkAQDw^^QFj%tA<4WKyRb)t)i=DV(urS#jq~7>0ow)|6REDH`H6J_`weBDjz(Hj9
zb$(bTgzk7e$6i|jOcEsBB?Zv(kaf?t*xV0hcUh#l`v>&6Khpi}ap>^S>t0U(%w!qJ
z?)FI8T}u|4yDQzpm7#&?bl?7MX9K(b-&eYyDNwzJI$@%zE&(O7w{|dz0Okl!$q=*q
zs4ff^jqWoWwAJ8b23hyw3-Hw4=svFx%S55O9?<gKb-hL4P92O6;D_Pgg9*N+`|%r~
z<?YlR#KpB{B_;#9I|B!DXLheakNYFtlMJE5gV!C5QwQgcB>|v24mV<UKWGEZ-IeY}
zPS8L&-N7)SCo{spy~C(=M^pR)VtG=04_Ts|%$)Rx!NTYcQhie`g-}ms<^$oWsC5UK
zpo*yH?%ct!ObFd`h7}Z1PiE}V^N@8P^-atGCo>dClNrVvXv0I$9c=FQ^tN*?fIAeL
zhhF#ithy6Az+^@aX)=?08=4EDI~XYRWai|3XdpV>p(iucDNs~2wK-t0%$*lHwg8Ke
zCo}F3VX$a)zqKG$3o)4~gs1LC_w`4xObFfCRpaI%CNq!G@{o1EHd9mc7|>k`c`}0@
z_eZ+-6+?%IUiUsh$5Z%#?iNUsnR9i}++FFO_yQUTr~3uKOK9C4TcD_DN{2w~yaMwv
zER=Ki&Nnbv7~Mgt;Jy>yV0K5I%uv6Dr|w2~odH-T3f;jNp><C{&)caxh$9+M&j7Ex
zBI_<Qh&DU~-9a97c0Vx;&4bq+<Q_S^agq$s{W6m7#naGS2;D&-bauyDga)G19lCPh
zEhH69WfK_eNTKr?B|!HGD#W>aDkcuZ90|U%38cCT_+R@D=nhl{B3zl_!-c1!)*WO*
zPiC&*!7?FqubO542;75>Aq&LF0K)%$rTgWh=OOD}bZw7aD}ee&fjD<J!ABb&3f-X#
zuWSjSdFXX#6B4|B0??fXX)-gqADX)>-F1#Y1L1TB*Lb0I=ca?8qAR=tvHptzB=RVA
zU*>|r!en=l%6Xbz99&gJ*8MskJasp^KjVjGqR<_T5n6Xl0kpiGx`ViwnF~HIQR@B_
zJq|&4kO!@M-Ervf;B^PNd2<YpMFHLSBk9gB3(bYl9RxyW_k3k&;O=z~P=TVNSylbn
zdV~if|ITr$dN5ctx+6a0^*6QO0G^6kcQ8_rf%;UH(h!#U$GU5w=OOEk_>9uuO(wpC
zHoQO5{h%>4554Y)PjmSlXNVm%cUQWTx<LcsbO%clw)ghVvb9MRREV-QRQtsLZZDlX
zQh3{Tjo-eM6aWN-xb66F&cUn59N^N??@h?Jqd=1px4Qm~n)5_PLGMF_c%1hC0s-&o
zeoJ1+EaZwSFdpcND>iSG3<O=l$}hw%oqvUr?l*|Y__e|F_QQf|)5YHiFJRv~)LSra
zXQ0~T@HgXZFb?9Dhln2rfB9F20XGx;^#V3)yRq}bKMw#daD5X4V-5J{{8^yX4EPlc
GjQ;~;Q{b!s

literal 0
HcmV?d00001

diff --git a/research/integration/activation_hit_target_v1/CONTROLS.json b/research/integration/activation_hit_target_v1/CONTROLS.json
new file mode 100644
index 0000000..cb36f3f
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/CONTROLS.json
@@ -0,0 +1,106 @@
+{
+  "controls": [
+    {
+      "case": 0,
+      "errors": [
+        "final_effect:a"
+      ],
+      "name": "final_value",
+      "rejected": true
+    },
+    {
+      "case": 3,
+      "errors": [
+        "callback_count"
+      ],
+      "name": "omitted_callback_reindexed",
+      "rejected": true
+    },
+    {
+      "case": 0,
+      "errors": [
+        "hit_chain:pre",
+        "tk_native_hit_agreement:pre",
+        "pre_hit_control",
+        "receipt_binding"
+      ],
+      "name": "wrong_hit_leaf",
+      "rejected": true
+    },
+    {
+      "case": 0,
+      "errors": [
+        "receipt_binding"
+      ],
+      "name": "receipt_hit_binding",
+      "rejected": true
+    },
+    {
+      "case": 5,
+      "errors": [
+        "receipt_only_decision"
+      ],
+      "name": "flipped_decision",
+      "rejected": true
+    },
+    {
+      "case": 0,
+      "errors": [
+        "native_count:2"
+      ],
+      "name": "omitted_key_reindexed",
+      "rejected": true
+    },
+    {
+      "case": 0,
+      "errors": [
+        "incomplete:KeyError:'app_exit'"
+      ],
+      "name": "missing_app_exit",
+      "rejected": true
+    },
+    {
+      "case": 0,
+      "errors": [
+        "app_exit"
+      ],
+      "name": "bool_app_exit",
+      "rejected": true
+    },
+    {
+      "case": 0,
+      "errors": [
+        "independent_neutral"
+      ],
+      "name": "non_neutral",
+      "rejected": true
+    },
+    {
+      "case": 0,
+      "errors": [
+        "native_geometry_invariant"
+      ],
+      "name": "changed_geometry",
+      "rejected": true
+    },
+    {
+      "case": 3,
+      "errors": [
+        "independent_marker:after_capture"
+      ],
+      "name": "marker_with_recomputed_hash",
+      "rejected": true
+    },
+    {
+      "case": 0,
+      "errors": [
+        "decision_time"
+      ],
+      "name": "reversed_decision_time",
+      "rejected": true
+    }
+  ],
+  "pass": true,
+  "rejected": 12,
+  "total": 12
+}
diff --git a/research/integration/activation_hit_target_v1/ENVIRONMENT.json b/research/integration/activation_hit_target_v1/ENVIRONMENT.json
new file mode 100644
index 0000000..515a7c2
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/ENVIRONMENT.json
@@ -0,0 +1,47 @@
+{
+  "benchmark": false,
+  "binaries": {
+    "Xvfb": {
+      "path": "/usr/bin/Xvfb",
+      "sha256": "4efd5850f0d98c587195ef77be92916dd1578fb725ddd1b2d95d232042813c5e"
+    },
+    "xauth": {
+      "path": "/usr/bin/xauth",
+      "sha256": "39e89b8530af1402b6b6f845309fc5a578c9ef9105412fa7198927ce92e7199d"
+    }
+  },
+  "clock": {
+    "adjustable": false,
+    "implementation": "clock_gettime(CLOCK_MONOTONIC)",
+    "monotonic": true,
+    "resolution": 1e-09
+  },
+  "cpu_affinity": [
+    0,
+    1,
+    2,
+    3,
+    4
+  ],
+  "cpu_model": "AMD EPYC 9V74 80-Core Processor",
+  "cpus": 5,
+  "docker": null,
+  "executable": "/opt/pyvenv/bin/python",
+  "executable_sha256": "17b78e0a93175e86f9ac03141924fd7a7f0c0c52e66b34bfa0de20ffef989df1",
+  "frequency_controlled": false,
+  "gh": null,
+  "image_attestation": null,
+  "platform": "Linux-6.18.44-x86_64-with-glibc2.41",
+  "python": "3.13.5 (main, Jul 15 2026, 20:25:40) [GCC 14.2.0]",
+  "python_xlib": [
+    0,
+    15
+  ],
+  "screen": [
+    640,
+    360,
+    24
+  ],
+  "tk_interpreter": "8.6.16",
+  "xlib_init_sha256": "2799b37f544bfadf7173eb64649ac2c7d2fbe1cf6fd8a8bba35b3da0836a9893"
+}
diff --git a/research/integration/activation_hit_target_v1/FREEZE.json b/research/integration/activation_hit_target_v1/FREEZE.json
new file mode 100644
index 0000000..e986a86
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/FREEZE.json
@@ -0,0 +1,21 @@
+{
+  "allocation": "activation-hit-target-20260922-01",
+  "formal_cases_at_freeze": 0,
+  "registration": "LOCAL_ONLY_BEFORE_FORMAL",
+  "sha256": {
+    "CONSTRUCTION.zip": "362b6da9e7acf41e533c406054fc3139e0eb8e1b95c3717080631fdbb6250ac9",
+    "ENVIRONMENT.json": "6a0e2b60641497092e0cb6d6ba5a9e0e1bad54f17d5031408631627002ef85b6",
+    "INTAKE.json": "19dc760d6efa506329005d437312a24116d3de2336cfb2236b817735fb864cf4",
+    "PLAN.md": "c05684dd64811b0b7b1f07f61be11d6f7b36c48d3bd3a8ada71c7a9305faf48c",
+    "app.py": "102e0d7584463ffb52e35a91d71696ae691a228be88081454a95c7f88c9d006d",
+    "audit.py": "4cdfc6ddada8551dd6d6e2d7389a228ab3bf6f59934b6a6bf9e266549352b2d7",
+    "backend.py": "3429a422e61ecb8b1f1f278540d0696842d8197d7967803e01bc8d9453bcb4a8",
+    "controls.py": "b99f4d44196be49184ea34e37f943945d2754bfb044bd518e08b6f22883dcaa4",
+    "launch.py": "1960d4eacb7b014e0f67c74b10dd19f46dd5211c9aba1b3b7aa6a56d4bcf05f6",
+    "load_backend.py": "cfc89e0e926bc6b4f3feae68d26235d7da810e064604fbddfeb9566d663cab43",
+    "policy.py": "9d2fe1684b655a8c9077f9e86043ee15acd992d568d76b170d218a59be806c03",
+    "runner.py": "52761419d1d61858498f9d57bf106e3a2081c72f782bbd5deaac98c3a9a36087",
+    "test_policy.py": "9a9bd6a68b49c467b3508a8341215fe1acdd709dee8e34dd69443035a3b862f7"
+  },
+  "utc": "2026-09-21T23:05:47.202827+00:00"
+}
diff --git a/research/integration/activation_hit_target_v1/INTAKE.json b/research/integration/activation_hit_target_v1/INTAKE.json
new file mode 100644
index 0000000..a0c8f3c
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/INTAKE.json
@@ -0,0 +1,33 @@
+{
+  "base_main": "2308b8301d69b7089a2e0636486736ed59b61537",
+  "branch_pages_counts": [
+    100,
+    38
+  ],
+  "goal_blob": "cb65e252e30b3552e960b1ee335ca6229be09d14",
+  "pr_head": "965cadf1636b00da30ab7044a55c8179c56b5ee6",
+  "preceding_experiment_issue": 4036,
+  "preceding_pr": 4053,
+  "predecessor_issue": 55,
+  "prior_attachments_sha256": {
+    "issue2547-additive.patch": "d59948a4e6923130291d341abcf605201332be89e227fe4db2804926697cfd05",
+    "issue2547-delivery-validation.json": "2e54d82865cdf3d3f4340a573dfb8dc662a55ca8fa41d6525b66a777594d8563",
+    "issue2547-report-ja.md": "c1ee938308d89384cb1ff37d8a18eaa03a02741fe041c257badd2ebb48c2808a",
+    "issue2547-research-bundle.zip": "f6fcf01e9280871b761990528cb279c9f32b6f56c8207bf95e42291ac4fa5787",
+    "scheduler-deadline-3934-additive.patch": "2e5114dbfa6f5ad53fdfb15035020a588187488a8e5af5e4039d2803e58bc9dd",
+    "scheduler-deadline-3934-delivery-validation.json": "b260f0f5012bdf18ced5db8e138d4b73f9c4f4570898709b17717ad5dcd9f803",
+    "scheduler-deadline-3934-report-ja.md": "2b6fdd89edd56bf9949d42d03502cfc32fd898a84eea96fe0cf68051a16fdbbb",
+    "scheduler-deadline-3934-research-bundle.zip": "c73c9066ddcb530b83ec93ae3b5e1e53f4acc361fd5796700fd4f026f410a42d"
+  },
+  "public_docs": [
+    "https://www.tcl-lang.org/man/tcl8.6/TkCmd/winfo.htm"
+  ],
+  "publication": "READ_ONLY_CONNECTION_NO_REMOTE_MUTATION",
+  "readme_blob": "7aa60f1c2e151724b0696a54b75267e62faa1787",
+  "roadmap_blob": "6c79de4666772d7bf3230a182d20631be6ff2d0b",
+  "scope": "bounded live connector search, unpushed work unknown",
+  "searches_no_exact_match": [
+    "activation-hit branch",
+    "activation_hit_target Issue"
+  ]
+}
diff --git a/research/integration/activation_hit_target_v1/ISSUE_PROPOSAL.md b/research/integration/activation_hit_target_v1/ISSUE_PROPOSAL.md
new file mode 100644
index 0000000..301aca1
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/ISSUE_PROPOSAL.md
@@ -0,0 +1,9 @@
+# UNPOSTED proposal: click-to-focus recovery with an occluded target
+
+Suggested lineage: successor scope to #4036 / PR #4053, related closed #55 and #2789. This is a different native hit-target/side-effect condition explicitly outside #4036, not an extension of its denominator. Check live ownership before publication; do not invent an Issue number or re-execute the allocation.
+
+This proposal records the already completed, locally preregistered `activation-hit-target-20260922-01`, not prospective public preregistration. Exact H/T/D/C/U is preserved in PLAN.md; first source freeze and all 26 native outcomes are included. Result: PASS_ACTIVATION_HIT_TARGET_BOUNDARY_SCOPED, confirming the bounded failure modes, not approving a universally safe candidate. CLICK_THEN_TYPE / POST_FOCUS / HIT_AND_FOCUS each have eight cases and produce 4/0/0 wrong-field texts and 4/4/2 unwanted Button callbacks respectively. All clear/unrelated-cover positives work; no task input in the two controls. See REPORT.md and AUDIT.json.
+
+One same-app stacked Button can intercept a focus-recovery click without changing the registered Entry's XID/geometry. Post-click recipient evidence protects text but not the earlier callback. A fresh native hit check rejects preexisting occlusion but not later occlusion. No general GUI/model/cost/atomicity claim follows.
+
+Current delivery is blocked locally by absent GitHub write capabilities. Keep that incident in this same evidence record. No shared code/earlier result/parallel branch was modified; no retries; no runtime promotion or global roadmap closure. This draft has NOT been posted.
diff --git a/research/integration/activation_hit_target_v1/ORDER_RECHECK.json b/research/integration/activation_hit_target_v1/ORDER_RECHECK.json
new file mode 100644
index 0000000..711b5ab
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/ORDER_RECHECK.json
@@ -0,0 +1,76 @@
+{
+  "all_ordered": true,
+  "rows": [
+    {
+      "after_click_capture_ns": 731952592097,
+      "callback_ns": 731942079831,
+      "case": 3,
+      "ordered": true,
+      "type_decision_ns": 731953053648
+    },
+    {
+      "after_click_capture_ns": 732781783518,
+      "callback_ns": 732771339522,
+      "case": 4,
+      "ordered": true,
+      "type_decision_ns": 732782360103
+    },
+    {
+      "after_click_capture_ns": 734439271936,
+      "callback_ns": 734428743255,
+      "case": 6,
+      "ordered": true,
+      "type_decision_ns": 734439836714
+    },
+    {
+      "after_click_capture_ns": 735245591199,
+      "callback_ns": 735235122757,
+      "case": 7,
+      "ordered": true,
+      "type_decision_ns": 735246078983
+    },
+    {
+      "after_click_capture_ns": 736049981275,
+      "callback_ns": 736039480225,
+      "case": 8,
+      "ordered": true,
+      "type_decision_ns": 736050573352
+    },
+    {
+      "after_click_capture_ns": 750875178936,
+      "callback_ns": 750864554084,
+      "case": 15,
+      "ordered": true,
+      "type_decision_ns": 750875668342
+    },
+    {
+      "after_click_capture_ns": 752568245679,
+      "callback_ns": 752557707254,
+      "case": 17,
+      "ordered": true,
+      "type_decision_ns": 752568757358
+    },
+    {
+      "after_click_capture_ns": 754869238727,
+      "callback_ns": 754858735308,
+      "case": 18,
+      "ordered": true,
+      "type_decision_ns": 754870259477
+    },
+    {
+      "after_click_capture_ns": 755827728289,
+      "callback_ns": 755817051704,
+      "case": 19,
+      "ordered": true,
+      "type_decision_ns": 755828676646
+    },
+    {
+      "after_click_capture_ns": 756701952764,
+      "callback_ns": 756691528802,
+      "case": 20,
+      "ordered": true,
+      "type_decision_ns": 756702452641
+    }
+  ],
+  "scope": "postformal_read_only_supplement"
+}
diff --git a/research/integration/activation_hit_target_v1/PLAN.md b/research/integration/activation_hit_target_v1/PLAN.md
new file mode 100644
index 0000000..051b34a
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/PLAN.md
@@ -0,0 +1,86 @@
+# Click activation: hit target versus keyboard recipient
+
+Allocation: `activation-hit-target-20260922-01`.
+Status at this document's freeze: zero formal cases. This is LOCAL preregistration only.
+Repository: Unjuno/agent-interface. Intake main: 2308b8301d69b7089a2e0636486736ed59b61537.
+Predecessors: closed #55; #4036 / open PR #4053 (head 965cadf1636b00da30ab7044a55c8179c56b5ee6).
+Ownership: additive `research/integration/activation_hit_target_v1/` only; proposed/local branch `research/activation-hit-target-20260922`. No remote branch or issue has been created.
+
+## Concrete integration decision
+
+Does an ordinary click used to recover the intended keyboard recipient remain a no-collateral recipe when a different native sibling covers the registered Entry? Separate hit-target evidence before button-down from recipient evidence after release. A successful focus call, unchanged target geometry and eventual neutral input are not task-effect evidence. This is a NEW occlusion/application-support scope explicitly excluded by #4036, not a rerun of its 30 cases. The earlier chat scheduler results remain immutable and are not measurement inputs here.
+
+## H
+
+H1: A registered Entry's native identity and geometry can remain unchanged while an ordinary same-application Button placed above it becomes the native pointer hit target. A geometry-only click followed by typing can invoke that Button and put the digit in the previously focused second Entry.
+H2: Checking the intended keyboard recipient after the click prevents that wrong-field text but does not undo the already committed Button callback.
+H3: Checking the native pointer hit target before clicking prevents a cover already present at that check, preserving clear and unrelated-overlay positives. A new cover between that check and button-down still permits the collateral callback; a later focus check only refuses the text. The candidate is NOT hypothesized to solve the remaining check/use interval.
+
+This is a finite application/backend characterization of known window-stacking behavior, not a security finding, new X11 theorem, full runtime admission result or general safe recipe.
+
+## T
+
+Environment: provided Linux x86_64 execution container, CPython 3.13.5, Tcl/Tk 8.6.16, installed Python-Xlib (0.15), Xvfb, five allowed guest CPUs. Exact binary/interpreter hashes in ENVIRONMENT.json. No Docker/OrbStack/image attestation, installation, external experiment network, model/provider, host desktop, user document, or clipboard use.
+
+Each batch uses an owned TCP-disabled Xvfb with a private generated MIT cookie; all native effects stay in that server. The authentication cookie is removed at cleanup and is not evidence to publish. Each case launches a fresh Tk application and has a different opaque session ID. Ordinary Entry and Button class bindings are unchanged. Fixture IPC only observes, repositions/maps the cover, or closes; it never inserts task text or directly invokes the callback. Input uses the exact current X11Backend methods. The complete vendored backend Git blob is 9cae101a219348077668c8fc086acf8e13154afe; SHA256 3429a422e61ecb8b1f1f278540d0696842d8197d7967803e01bc8d9453bcb4a8. Loader removes exactly the unused runtime.core_v1.contract import; no method body is rewritten. manifest(), core admission, full execute()/CLI/MCP are not exercised.
+
+Three policies:
+- CLICK_THEN_TYPE: common registered-surface focus + pointer move, then click and type the digit.
+- POST_FOCUS: same click, but type only with current application-recipient evidence for A.
+- HIT_AND_FOCUS: first require the separate X observer's deepest pointer child to be A; then require the same post-click recipient check.
+
+Four schedules: CLEAR; COVER_BEFORE (ordinary cover above A before hit observation); COVER_AFTER (cover only after the hit decision, before click); UNRELATED (same Button elsewhere). Two rotations of policy order yield 24 fresh cases. Two additional NO_TASK_INPUT controls, CLEAR and COVER_BEFORE, retain the common focus/motion but no button or key input. They are NOT zero-OS-input controls. Minimum denominator: 26, not a sample-size estimate.
+
+Use three fixed serial batches of indices 0..8, 9..17, 18..25. Only one invocation per formal batch. Each case has a 12-second application watchdog, bounded 3-second RPC response, 2-second native-event observation deadline, and verified native release. The outer supervisor waits at most 35 seconds per batch and records the real child wait status. No new batch follows a nonzero/missing previous exit or incomplete prior END. Preserve timeout/partial state; no rerun, replacement, pooling with construction, postfreeze tuning, or automatic overlay removal.
+
+Source, independent audit, controls, unit tests, environment, schedule, expected table and construction evidence are hash-frozen locally before formal batch 0. Exact formal commands:
+```
+python -B launch.py formal <study>/results/formal-batch-00 --batch 0
+python -B launch.py formal <study>/results/formal-batch-01 --batch 1
+python -B launch.py formal <study>/results/formal-batch-02 --batch 2
+```
+The runner enforces its own exact source-relative output roots; existing roots refuse. These are recorded reproduction commands, not permission to repeat the retained allocation.
+
+## D
+
+Expected per-policy totals over eight cases:
+| Policy | Correct A values | Wrong B values | Cover callbacks |
+|---|---:|---:|---:|
+| CLICK_THEN_TYPE | 4 | 4 | 4 |
+| POST_FOCUS | 4 | 0 | 4 |
+| HIT_AND_FOCUS | 4 | 0 | 2 |
+The two no-task-input controls remain blank with zero callbacks. Every CLEAR/UNRELATED positive must have A='7', B='', counter=0. HIT_AND_FOCUS refuses both COVER_BEFORE cases before button-down. All COVER_AFTER cases with a click retain the one actual callback, even when the text is refused. These are scope failures for unconditional recovery, NOT safe-candidate successes.
+
+PASS_ACTIVATION_HIT_TARGET_BOUNDARY_SCOPED requires the complete 26-case/3-batch denominator; exact source and process/command/event/effect identity; unchanged target geometry; native and Tk hit-test agreement; exact ordinary input events; independent marker-pixel reconstruction; all release/keymap/button/server/exit gates; zero audit errors; and all 12 frozen copied-evidence mutations rejected. Unit tests require seven successes. Any complete contrary pattern is a hypothesis mismatch; absent/corrupt provenance, process or cleanup evidence is HOLD/STOP. The auditor uses the conservative HOLD_EVIDENCE_OR_GATE_FAILURE envelope for either class and retains exact causes for the report. A boundary PASS does not approve either weak comparator or imply that HIT_AND_FOCUS prevents after-check collateral effects.
+
+## C
+
+A cooperative synthetic app supplies exact Entry identity/geometry and post-click focus. The pointer observer uses a separate native X connection and walks QueryPointer child results; the Tk winfo-containing response is independent cross-checking evidence. Multiple queries and subsequent input are not atomic. There is no popup grab, foreign process overlay, focus change after the post-click check, concurrent geometry edit, disabled field, text selection, IME, window reincarnation, or production semantic target discovery. Two app-observation times are not collapsed into one atomic snapshot claim. The cover's local counter is a harmless private fixture effect, not a user action.
+
+## U
+
+Two directed repetitions are finite coverage, not natural error probability or calibrated reliability. No performance gate, model latency, token reduction, broad correctness, human tempo or runtime promotion. Clock values are diagnostics in the same monotonic domain; X-server millisecond event times are retained but never subtracted from host nanoseconds. CPU frequency/load is uncontrolled. Combined physical measurement uncertainty u_c and coverage factor k are NOT estimated or applicable to this logical contract verdict.
+
+## Variable / field table
+
+| Field | Meaning | SI unit | Definition / domain | Type |
+|---|---|---|---|---|
+| target / surface / hit / focus | Target, surface, native hit child and Tk recipient identities | 1 | Positive native XID; absent recipient can be null | integer scalar / null |
+| geometry | Target root position and extent | 1 (logical pixels, not metres) | x,y integer; width,height positive; frozen 200 by 40 | four-component integer record |
+| session | One fresh app lifetime | 1 | Opaque 32-hex identifier supplied to the app | string |
+| sequence | Receipt order within one case | 1 | Positive integer | integer scalar |
+| captured_ns / decided_ns | Same-container monotonic observation/decision instants | s, stored in ns | nonnegative integer; ordering only, no physical calibration | integer scalar |
+| allow | This fixture recipe's next-step eligibility | 1 | true/false, not production authority | Boolean |
+| counter | Application Button callback count | 1 | integer initially 0; increments only in ordinary callback | integer scalar |
+
+Unit check: identities, pixel coordinates and event counts are never compared with durations. All temporal order checks compare host nanoseconds within the same domain. No assertion equates X-server logical neutral state with physical hardware state.
+
+## Roadmap and publication
+
+Intake/lineage/collision search -> excluded construction and correction -> exact local source freeze -> three one-shot native batches -> separate raw audit and 12 controls -> lossless evidence, report and additive patch -> GitHub issue/PR/checks/readback when a supported write route exists -> own-branch cleanup only after verified merge/dependency checks.
+
+Current connector discovery exposes 48 read operations and no write action. Plugin search returned the already installed GitHub connector only. Local gh/Docker and GH_TOKEN/GITHUB_TOKEN are absent. This local publication constraint is NOT a global fleet blocker or a new scientific question. Record STOP_PUBLICATION_NO_WRITE_CAPABILITY under this delivery, without inventing a remote Issue/PR number. Public prior results, parallel branches and global ROADMAP remain unchanged.
+
+## Related fields
+
+HCI: click recovery can have effects other than focus. Distributed/concurrent systems: a precheck is not atomic with use. Software verification/metrology: input release, recipient identity and independently observed application effect require separate evidence. These are transfer ideas, not evaluated additional domains.
diff --git a/research/integration/activation_hit_target_v1/PR_DRAFT.md b/research/integration/activation_hit_target_v1/PR_DRAFT.md
new file mode 100644
index 0000000..88eab00
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/PR_DRAFT.md
@@ -0,0 +1,9 @@
+# UNPOSTED PR draft — research: separate activation hit target from keyboard recipient
+
+Add only `research/integration/activation_hit_target_v1/**`. Preserve #4036/PR #4053 and all old allocations; close no broad parent.
+
+Retrospective delivery of one locally preregistered native experiment, not public preregistration. The 26-case/3-batch allocation passed its frozen boundary gates; 3,094 independent raw checks, 12/12 evidence mutations and seven policy tests pass. All actual app/server/runner exits are zero. Native backend blob is exact; only an unused manifest import is omitted during research loading.
+
+Main result: checking focus AFTER a recovery click stops wrong text but can leave a committed collateral Button effect. Checking the native hit child BEFORE the click protects already-occluded targets but still does not solve the after-check interval. Do not label the candidate generically safe. Include all four construction attempts, including the authentication STOP; no formal rerun or result rewriting.
+
+Delivery patch is additive and scratch-application-tested, but upstream repository CI/review has not run. Before merging, verify target path collision, source/evidence hashes, current-head checks and compatibility with the result/recovery spine. No shared runtime/default change is proposed. Remove only the newly owned branch after a verified merge and dependency review, using supported tooling. This document is not evidence that a PR exists.
diff --git a/research/integration/activation_hit_target_v1/README.md b/research/integration/activation_hit_target_v1/README.md
new file mode 100644
index 0000000..48515d8
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/README.md
@@ -0,0 +1,27 @@
+# Activation hit-target boundary — retained local research
+
+Read REPORT.md for the result and PLAN.md for the locally frozen H/T/D/C/U, exact schedule, variable table, limits and stop rules.
+
+This additive research bundle records one actual 26-case private-Xvfb/Tk allocation. It does not change shared runtime, grant production authority, or claim GitHub publication. No model or user-desktop input occurred.
+
+## Safe offline review
+
+These commands only inspect retained records; they do not start a display or send input:
+
+```sh
+python -B audit.py .
+python -B controls.py results/formal-batch-00
+python -B -m unittest -v test_policy
+```
+
+The raw-only audit requires Python standard library only. The unit tests import only the receipt policy. `load_backend.py` verifies the exact upstream blob before loading and is used only by the live runner, not by audit. The policy alone is not a production admission guard.
+
+Do not rerun `launch.py formal` or delete the consumed result roots to make it run. All three exact roots and their true exit receipts are present. Construction evidence is archived separately, including the first authentication failure. Raw screenshots are compressed original XImage byte arrays; descriptors retain format/masks/width/height/hash. `examples/` images, when present, are explicitly derived display aids, not new observations.
+
+SHA256SUMS.json covers every publication member except itself. Validate it before review. Upstream source identity is additionally fixed by the original FREEZE.json and Git blob.
+
+## Publication status
+
+The proposed branch is `research/activation-hit-target-20260922`. It exists locally only, as an isolated root with no upstream ancestry. The delivery patch contains additions under this directory only. Applying the patch to an empty scratch repository is not upstream CI or a merge test. Existing concurrent branches and earlier evidence were not changed or deleted.
+
+Potential future publication must call this retrospective GitHub delivery of a locally preregistered experiment. No new scientific execution is needed to publish these bytes. Any new application hypothesis requires separate ownership and allocation checks; a transport/tool failure does not create a new research objective.
diff --git a/research/integration/activation_hit_target_v1/READ_ONLY_RECHECK.json b/research/integration/activation_hit_target_v1/READ_ONLY_RECHECK.json
new file mode 100644
index 0000000..56c05ae
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/READ_ONLY_RECHECK.json
@@ -0,0 +1,34 @@
+{
+  "audit_byte_identical": true,
+  "audit_exit": 0,
+  "audit_stderr": "",
+  "callback_order_recheck": true,
+  "frozen_sources": {
+    "CONSTRUCTION.zip": true,
+    "ENVIRONMENT.json": true,
+    "INTAKE.json": true,
+    "PLAN.md": true,
+    "app.py": true,
+    "audit.py": true,
+    "backend.py": true,
+    "controls.py": true,
+    "launch.py": true,
+    "load_backend.py": true,
+    "policy.py": true,
+    "runner.py": true,
+    "test_policy.py": true
+  },
+  "prior_attachments_unchanged": {
+    "issue2547-additive.patch": true,
+    "issue2547-delivery-validation.json": true,
+    "issue2547-report-ja.md": true,
+    "issue2547-research-bundle.zip": true,
+    "scheduler-deadline-3934-additive.patch": true,
+    "scheduler-deadline-3934-delivery-validation.json": true,
+    "scheduler-deadline-3934-report-ja.md": true,
+    "scheduler-deadline-3934-research-bundle.zip": true
+  },
+  "unit_count": 7,
+  "unit_exit": 0,
+  "unit_stderr": "test_bool_hit_focus (test_policy.PolicyTests.test_bool_hit_focus) ... ok\ntest_bool_ids (test_policy.PolicyTests.test_bool_ids) ... ok\ntest_changed_session (test_policy.PolicyTests.test_changed_session) ... ok\ntest_clear_positives (test_policy.PolicyTests.test_clear_positives) ... ok\ntest_control (test_policy.PolicyTests.test_control) ... ok\ntest_hit_then_focus (test_policy.PolicyTests.test_hit_then_focus) ... ok\ntest_no_scenario_input (test_policy.PolicyTests.test_no_scenario_input) ... ok\n\n----------------------------------------------------------------------\nRan 7 tests in 0.000s\n\nOK\n"
+}
diff --git a/research/integration/activation_hit_target_v1/REPORT.md b/research/integration/activation_hit_target_v1/REPORT.md
new file mode 100644
index 0000000..eb5ec66
--- /dev/null
+++ b/research/integration/activation_hit_target_v1/REPORT.md
@@ -0,0 +1,68 @@
+# Activation recovery: pointer hit target and keyboard recipient are separate
+
+**Result: PASS_ACTIVATION_HIT_TARGET_BOUNDARY_SCOPED.** This is a finite boundary finding, NOT approval of an unconditionally safe recovery recipe.
+
+Allocation `activation-hit-target-20260922-01` completed all 26 fresh Tk cases in three predeclared batches of 9/9/8. Formal batch invocations: 3; reruns/replacements/exclusions: 0. All 26 application exits and all three server/runner exits were observed as zero. The separate raw-only auditor passed 3,094 checks with zero errors; all 12 copied-evidence controls rejected; seven policy unit tests passed. These are independent implementations/processes by the same author, not external human review or another-machine replication.
+
+## Main result
+
+Each ordinary recipe has eight cases: two repetitions of clear, cover-before-check, cover-after-check, and unrelated-cover schedules. Two additional no-task-input controls retain common focus/motion but no click or key.
+
+| Recipe | Correct A text | Wrong B text | Unwanted cover callback | Cases |
